---
layout: home
---

<p>
<b>How do cells establish identities and make cell-fate decisions?</b><br><br>

The computational trans-regulatory biology group led by <a href="http://sydney.edu.au/science/people/pengyi.yang.php">Dr. Pengyi Yang</a> 
at <a href="http://sydney.edu.au/charles-perkins-centre">Charles Perkins Centre (CPC)</a>, University of Sydney, focuses on 
understanding the trans-omics networks that cut across signalling, transcriptional, translational and (epi)genomic 
regulations in controlling cell identities and cell-fate decision-making.
<br><br>
Molecular trans-regulatory programs comprised of cell signalling, transcriptional, translational, and epigenomic networks are 
central to health and disease. Computational approaches are instrumental in characterising trans-regulatory programs of cells 
at the systems level.
<br><br>
Our research lies at the interface of bioinformatics and systems biology. We develop computational and statistical models 
to reconstruct signalling cascades, epigenomics, transcriptional, and proteome networks, and 
characterise their cross-talk and trans-regulations in various cellular processes and systems. 
By integrating heterogeneous -omics data with the goal of generating testable hypotheses and predictions, 
our research contributes to the comprehensive understanding of trans-omic networks that underlie cellular homeostasis, 
proliferation, differentiation, and cell-fate decisions.

<!-- 
Systematic study of complex interactions in biological networks at a global viewpoint allows us to discover 
fundamental principles that are not intuitive and to uncover global properties that can only be discovered by 
integrating interactions between individual components. We develop computational methods and statistical models to 
reconstruct and characterise signaling cascades, and epigenomic and transcriptional networks that underlie cellular 
homeostasis, proliferation, differentiation, and cell-fate decisions. By integrating heterogeneous “–omics” data with the goal of generating testable hypotheses and predictions, our 
researach contribute to the comprehensive understanding of the cross-talk among cell signaling, 
epigenomic, and transcriptional regulations. 
-->

<IMG SRC="images/projects/BigPicture.png" ALIGN="RIGHT" hspace="5" vspace="5" HEIGHT="300">
<br>

<hr>

<p>
Our research focuses on developing machine learning algorithms and statistical models for
<ul>
    <li>analysis of mass spectrometry (MS)-based:</li>
    <ul>
        <li>proteomics data</li>
        <li>phosphoproteomics data</li>
    </ul>
    <li>analysis of transcriptomics and epigenomics data including:</li>
    <ul>
        <li>bulk and single-cell RNA-seq</li>
        <li>ChIP-seq and DNaseI</li>
        <li>Hi-C and ChIA-PET</li>
    </ul>
    <li>and their cross-talk for comprehensive understanding of trans-omic networks.</li>
</ul>
</p>
