---
layout: home
---

<p>
<b><font size="+2">Research Summary</font></b><hr>

Pengyi Yang, Ph.D., heads the <a href="https://sydney.edu.au/charles-perkins-centre/our-research/research-groups/computational-trans-regulatory-biology-group.html">
Computational Trans-Regulatory Biology group</a> at Charles Perkins Centre (CPC), the University of Sydney, 
and holds a conjoint appointment as Group Leader of Computational Systems Biology group at Children's Medical Research Institute (CMRI), 
at the Westmead Research Hub.<br>

Molecular trans-regulatory networks (TRNs) comprised of cell signalling, transcriptional, translational, and (epi)genomic regulations are 
central to health and disease. Computational approaches are instrumental in characterising TRNs of cells at the systems level. 
Our research lies at the interface of bioinformatics and systems biology. We develop computational and statistical models 
to reconstruct signalling cascades, epigenomics, transcriptional, and proteome networks, and 
characterise their cross-talk and trans-regulations in various cellular processes and systems.
By integrating heterogeneous trans-omic data with the goal of generating testable hypotheses and predictions, 
we aim to tackle the following research questions:<br>

<ul>
	<li>How do different layers of regulations talk to each other in controlling stem cell fate?</li>
    <li>Can we accurately predict stem cell differentiation trajectories based on their TRNs?</li>
    <li>The mechanisms of stem/progenitor cells in establishing identities and making cell fate decisions</li>
</ul>


<br>

<!-- 
Systematic study of complex interactions in biological networks at a global viewpoint allows us to discover 
fundamental principles that are not intuitive and to uncover global properties that can only be discovered by 
integrating interactions between individual components. We develop computational methods and statistical models to 
reconstruct and characterise signaling cascades, and epigenomic and transcriptional networks that underlie cellular 
homeostasis, proliferation, differentiation, and cell-fate decisions. By integrating heterogeneous “–omics” data with the goal of generating testable hypotheses and predictions, our 
researach contribute to the comprehensive understanding of the cross-talk among cell signaling, 
epigenomic, and transcriptional regulations. 
-->

<IMG SRC="images/projects/BigPicture.png" hspace="10" vspace="10" HEIGHT="400">

<hr>

<!--
<p>
<b>Openings</b>
There are two PhD scholarships available for research in Complex Biological Data Extraction. Please refer to the link below for more details:
<br>
http://sydney.edu.au/scholarships/research/faculty/science.shtml#BIODATA
</p>
-->

<p>
<b>Methodological innovation</b><br><br>

Our group are experienced in developing machine learning algorithms and statistical models for 
analysing the following types of data:
<ul>
    <li>Mass spectrometry (MS)-based:</li>
    <ul>
        <li>Proteomics</li>
        <li>Phosphoproteomics</li>
        <li>Lipidomics</li>
        <li>Metabolomics</li>
        <li>Redox proteomics</li>
    </ul>
    <li>Next-generation sequencing (NGS)-based:</li>
    <ul>
        <li>Bulk RNA-seq and microarray</li>
        <li>Single-cell RNA-seq</li>
        <li>ChIP-seq and RIP-seq of DNA/RNA binding proteins</li>
        <li>RNA Polymerase II, Histones and DNaseI</li>
        <li>Hi-C and ChIA-PET</li>
    </ul>
    <li>and their cross-talk for comprehensive understanding of TRNs in controlling cell identities and cell-fate decisions.</li>
</ul>
</p>




