---
layout: page
---
<h1 class="projects">    Projects</h1>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 1 illustration" src="../images/project1.png"/ Align="Left">
    </div>
    <div class="col-md-8">
        <h3 class="heading">Modeling the impact of genetic variation on multiple types of cellular traits</h3>
        <div class="content">
            We are developing integrative computational approaches for predicting the impact of genetic variation 
            on multiple types of cellular traits, including DNA methylation, histone modification,and gene expression.
            
            <br>
            People: Bernard Ng, Farnush Kiadeh
            <br>
            Alumni: Rosemary McCloskey, James Topham
            <br>
            Relevant publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24092820">here</a> and 
            <a href="http://www.ncbi.nlm.nih.gov/pubmed/24786080">here</a>
        </div>
    </div>
</div>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 2 illustration" src="../images/deconv.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Deconvolution of hidden and known confounding factors</h3>
        <div class="content">
            We are developing machine learning methods for modeling the effect of hidden and "partially known" confounding factors in gene expression and methylation data. 
            In this direction, we are currently developing methods for estimating cell type proportions in "genomics" samples from varied tissues including brain and cord blood.
            
            <br>
            Peaple: Louie Dinh
            <br>
            Relevant publications: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23874524">here</a>
        </div>
    </div>
</div>
<div class="section row">
    <div class="col-md-4 image">
        <img class="img-responsive" alt="Project 3 illustration" src="../images/mkm.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Multi omics analysis of complex disease</h3>
        <div class="content">
            Working with clinical colleagues at UBC and elsewhere, we are developing association analysis approaches using multiple types of omics data. 
            Specifically, we are currently focusing on integrative analysis of genotype, gene expression, DNA methylation, and histone modification
            data in order to identify genes and networks that are associated with neurodegenerative diseases (including AD & PD).
            <br>
            People: Bernard Ng, Halldor Thorhallsson
        </div>
    </div>
</div>
<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 4 illustration" src="/images/projects/AdaSampling.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Identification of transcription factor target genes using machine learning</h3>
        <div class="content">
            We have recently developed an adaptive sampling approach for learning from positive-unlabeled dataset. 
            We are extending this semi-supervised learning approach for the identification of transcription factor target
            genes in differentiating mouse embryonic stem cells (mESCs) by integrating transcriptomic and epigenomic data.             
            <br>
            People: Dinuka Perera, Chendong Ma
            <br>
            Relevant publications: <a href="http://www.ijcai.org/proceedings/2017/0457.pdf">here</a>
        </div>
    </div>
</div>
