---
layout: default
---

<div class="container">

<div class="section row">
	<div class="col-md-12">
	<h2 class="heading"><b>Research summary</b></h2><br>
        Cells are the fundamental building blocks of all life on earth. In multicellular organisms, these cellular building blocks come in a plethora of types and serve distinct functions to coordinate and support the homeostasis of an organism. Intriguingly, all cell types of an organism share the same genetic code and originate from the same group of pluripotent <b>stem cells</b> that have undergone different cellular fates during development.
        <br><br>
        Research from the last few decades has established that cell identity and cell-fate decisions are determined by underlying molecular programs such as gene transcription and translation, and the intra- and extra-cellular networks they form. Understanding how such programs/networks regulate and interact with one another in a spatial-temporal manner is critical for gaining insight into the complex nature of multicellular development. To this end, our research takes a holistic approach and seeks to understand the molecular 'trans-omic networks', comprised of cell signalling, transcriptional, translational, and (epi)genomic regulations, in controlling cell identity and cell-fate decisions. In particular, we develop computational and statistical models to reconstruct each layer of the trans-omic networks (e.g. signalling networks, transcriptional networks) and characterise their cross-talk in stem cells during their differentiation to specialised cell types. By employing a multidisiciplinary approach that combines 'dry' (computation) and 'wet' (laboratory) studies at the systems level, we aim to address the following research questions:<br><br>

        <ul>
            <li>How do different layers of trans-omic networks coordinately regulate cell identity and cell fate-decisions?</li>
            <li>How can we accurately predict cell identity and cell fate-decisions based on their trans-omic networks?</li>
            <li>How can we modulate trans-omic networks to direct cell fate-decisions for applications such as stem cell therapy?</li>
        </ul>
	</div>
</div>        
<hr>

<h2 class="projects"><b>Mapping and modelling trans-omic networks in stem cells and progenitor cells for understanding development, tissue regeneration, and stem cell therapy</b></h2>

The ability of stem and progenitor cells to self-renew and differentiate into specialised cell types in the body makes them the key cellular tools for studying normal development, developmental diseases, and for creating stem cell-based therapy, whereby specific cell populations, tissues, and organs compromised by disease, injury or ageing can be replaced and replenished indefinitely.<br><br>

A major initiative in our group is to integrate multi-omics datasets generated by state‐of‐the‐art mass spectrometer (MS) and next generation sequencer (NGS) from various stem and progenitor cells and during their differentiation to specialised cell types. The goal is to reconstruct trans-omic networks and to understand how different regulatory machineries (e.g. signalling, transcription, and epigenomics) cooperate to define cell types, functions, and fates, and to translate these knowledge for directed cell type differentiation and tissue/organ regeneration.

<div class="section row">
    <div class="image col-md-5">
        <img class="img-responsive" alt="Project 1 illustration" src="/images/projects/TransReg.png"/ Align="Left">
    </div>
    <div class="col-md-7">
        <h3 class="heading">Reconstructing trans-omic networks in mouse embryonic stem cells (ESCs)</h3>
        <div class="content">
            Pluripotency defines the potential of pluripotent stem cells (PSCs) to differentiate into all somatic and germ cells. We have previously profiled the multi-layered trans‐omics networks in ESCs during their pluripotency transition (<a href="https://doi.org/10.1016/j.cels.2019.03.012">Yang et al. 2019</a>). Building on this work, we are developing methods to characterise signaling cascades, transcriptional networks, and translational protein networks in regulating the transition of pluripotency in ESCs, and to uncover how information is passed on from each of these regulatory layer to the next layer. This project will provide a unique opportunity for developing computational and experimental methods for comprehensive understanding of stem cell systems and their decision-making process.
        </div>
    </div> 

   <div class="col-md-12">     
      <h3 class="heading">'Mini organs' in a dish: identifying molecular controls in generating brain and retina organoids from human PSCs</h3>
      <div class="content">
          While tremendous progress has been made in stem cell therapy, attempts to direct stem/progenitor cells to clinically useful cell types have often been fraught with imprecise and incomplete differentiation. In collaboration with Dr. Anai Gonzalez Cordero's group at CMRI, we are reconstructing trans-networks networks and identifying key network modules and nodes that underlie the generation of brain and retina organoids using human PSCs. Discovery made from these works will have provide both basic and translational knowledge for directed cell type differentiation and tissue/organ regeneration.    
      </div>
   </div>
    
    <div class="col-md-12">
       <h3 class="heading">Identifying molecular controls in muscle cell formation</h3>
       <div class="content">
           In collaboration with Dr. Benjamin Parker's group at the Melbourne University, we are identifying molecular controls that regulate the differentiation of progenitor cells to mature muscle cells. To do this, we have profiled the trans-omic networks on phosphoproteomics and proteomics levels and have developed various computational methods allowing the identification of key phosphorylation events and their downstream transcriptional regulation in controlling myogenesis, formation of muscle cells. Finding from this study will have implications in muscle tissue regeneration.      
      </div>
   </div>    

</div>
<hr>


<h2 class="projects"><b>Interrogating cell identity and cell-fate decisions in spatial contexts and using single-cell multi-omics</b></h2>

The advance of single-cell omics and its recent development towards multimodality, that allows profiling of gene expression, chromatin state, protein abundance and spatial location, creates unprecedented opportunities to study complex biological systems at resolutions that were previously unattainable. Our group is harnessing the power of these new biotechnologies to study how each single cell acquires its identity in organs and tissues and makes decisions during cell differentiation and embryogenesis.

<div class="section row">
    <div class="image col-md-10">
        <img class="img-responsive" alt="Project 2 illustration" src="/images/projects/scMerge.png"/>
    </div>
    <div class="col-md-7">
        <h3 class="heading">Creating a retina cell type altas using single-cell data acquired from primary retina tissues and retina organoids</h3>
        <div class="content">
            We are collaborating with Dr. Anai Gonzalez Cordero's group at CMRI to create the first retina cell type atlas by integrating single-cell omics data generated from various primary retina tissues and retina organoids. From this atlas, we uses our recently developed Cepo algorithm to derive a cell identity score for each gene with respect to the cell types in the atlas (<a href="https://doi.org/10.1038/s43588-021-00172-2">Kim et al. 2021</a>). These cell identity scores provide a basis for annotating cell types and assessing data quality and disease status in future single-cell omics data generated from retina tissues and retina orgnoids.
        </div>
    </div>
    
    <div class="col-md-12">
        <h3 class="heading">Identifying cells that show bipotential differentiation capacity during early embryogenesis</h3>
        <div class="content">
			This project involves experimental profiling of single cells in embryos during their differentiation to multiple cell lineages. Together with Prof. <a href="https://www.cmri.org.au/Research/Research-Units/Embryology/Our-People">Patrick Tam's lab</a>, we are developing experimental and computational techniques for uncovering cells that could differentiate into multiple lineages during early embryogenesis. The results from this study will provide new knowledge to cell lineage specification in embryogensis and shed light on early developmental diseases	
        </div>
    </div>
    
</div>
<hr>

<h2 class="projects"><b>Developing computational methods for pre-processing, analysing, and integrating bulk and single-cell multi-omics data and their spatial information</b></h2>

Much of the work in the above research direction requires computational methods that are capable of extracting biological knowledge from large-scale of omic data generated from bulk and single-cell omics technologies. To enable these biological interrogation and discoveries, we specialise in developing computational and statistical machine learning models for analysing and integrating these data.
    
<div class="section row">
	<div class="col-md-12">
        <h3 class="heading">Machine learning methods for cell type annotation using single-cell multi-omic data</h3>    
        <div class="content">
            Cell type identification is a key step in single-cell omics experiments. Our previous work has demonstrated that machine learning could be effectively applied to annotate cell types using their transcriptomic profiles <a href="https://doi.org/10.15252/msb.20199389">Lin et al.</a>. Building on this work, we are in process of developing deep learning-based methods for annotating cell types and identifying molecular features associated with cell type annotation from single-cell multi-omics data.
        </div>
    </div>

    <div class="col-md-12">
        <h3 class="heading">Estimating the number of cell types from single-cell omic data</h3>
        <div class="content">
			Identifying the number of cell types in a biological system is an essential task prior in cell type annotation. Building on our previous work that uses an ensemble of deep learning models for cell type clustering <a href="https://doi.org/10.1186/s12859-019-3179-5">Geddes et al.</a>, we have recently implemented a stability based approach for number of cell type estimation <a href="https://doi.org/10.1186/s13059-022-02622-0">Yu et al.</a> from single-cell transcriptomics data. We are now exploring their application in single-cell multi-omics data.
        </div>
    </div>
</div>
<hr>

    
<div class="section row">
	<div class="col-md-12">
	<h2 class="heading"><b>Openings</b></h2>

<b>Post-doctoral positions</b> funded by NHMRC grant are available for conducting research on the broad area of computational systems biology in stem cells and their therapeutic application. Interested candidates are encouraged to please contact Pengyi Yang, lab head, to discuss potential projects and other details (<a href="mailto:pengyi.yang@sydney.edu.au">pengyi.yang [at] sydney.edu.au</a>)
<br><br>
<b>PhD scholarships</b> are available for both domestic and international candidates. Please see <a href="https://www.cmrijeansforgenes.org.au/research/opportunities-for-research-students/cmri-phd-research-award">link</a> for more details on Children's Medical Research Institute PhD Research Award. For details on scholarships offered at the University of Sydney, please see <a href="https://www.sydney.edu.au/scholarships/">link</a>.
<br><br>
<b>Honours projects</b> are available through either Children's Medical Research Institute, Faculty of Health and Medicine, see <a href="https://www.sydney.edu.au/science/about/our-people/academic-staff/pengyi-yang.html#collapseprofilehonorsopportunities">link</a>; or the School of Mathematics and Statistics, Faculty of Science. For more details regarding projects, please contact <a href="mailto:pengyi.yang@sydney.edu.au">Pengyi Yang</a>.
<br><br>
<b>Summer research scholarships</b> are available for third year undergraduates at both Children's Medical Research Institute and Charles Perkins Centre. For more details regarding potential projects and scholarships, please contact <a href="mailto:pengyi.yang@sydney.edu.au">Pengyi Yang</a>.
<br><br>
	
	For postgraduate applicants, all 'wet' projects require experience with cell culturing and good understanding of basic experimental protocols (e.g. qPCR, westernblotting) and preferably experience of (single cell-)sequencing and mass spectrometry-based proteomics. All 'dry' projects require programming skill in at least one programming/scripting language (R, Perl, Python, Java, C++, C and Matlab). There are ample opportunity to work closely with biologists, statisticians, bioinformaticians, and computer scientists.
	<br><br>
	More information about scholarship opportunity at the University of Sydney can be found <a href="https://sydney.edu.au/study/study-options/postgraduate-research.html">here</a>.<br>
	For obtaining scholarships and top-ups from CMRI, please read <a href="https://www.cmrijeansforgenes.org.au/research/opportunities-for-research-students/cmri-phd-research-award">here</a>.	
	</div>
</div>    

</div>
















