---
layout: default
---

<div class="container">

<div class="section row">
	<div class="col-md-12">
	<h2 class="heading"><b>Research summary</b></h2><br>
        Cells are the fundamental building blocks of all life on earth. In multicellular organisms, these cellular building blocks come in a plethora of types and serve distinct functions to coordinate and support the homeostasis of an organism. Intriguingly, all cell types of an organism share the same genetic code and originate from the same group of pluripotent <b>stem cells</b> that have undergone different cellular fates during development.
        <br><br>
        Research from the last few decades has established that cell identity and cell-fate decisions are determined by underlying molecular programs such as gene transcription and translation, and the intra- and extra-cellular networks they form. Understanding how such programs/networks regulate and interact with one another in a spatial-temporal manner is critical for gaining insight into the complex nature of multicellular development. To this end, our research takes a holistic approach and seeks to understand the molecular 'trans-omic networks', comprised of cell signalling, transcriptional, translational, and (epi)genomic regulations, in controlling cell identity and cell-fate decisions. In particular, we develop computational and statistical models to reconstruct each layer of the trans-omic networks (e.g. signalling networks, transcriptional networks) and characterise their cross-talk in stem cells during their differentiation to specialised cell types. By employing a multidisiciplinary approach that combines 'dry' (computation) and 'wet' (laboratory) studies at the systems level, we aim to address the following research questions:<br><br>

        <ul>
            <li>How do different layers of trans-omic networks coordinately regulate cell identity and cell fate-decisions?</li>
            <li>How can we accurately predict cell identity and cell fate-decisions based on their trans-omic networks?</li>
            <li>How can we modulate trans-omic networks to direct cell fate-decisions for applications such as stem cell therapy?</li>
        </ul>
	</div>
</div>        
<hr>

<h2 class="projects"><b>Mapping and modelling trans-omic networks in stem cells and progenitor cells for understanding development, tissue regeneration, and stem cell therapy</b></h2>

The ability of stem and progenitor cells to self-renew and differentiate into specialised cell types in the body makes them the key cellular tools for studying normal development, developmental diseases, and for creating stem cell-based therapy, whereby specific cell populations, tissues, and organs compromised by disease, injury or ageing can be replaced and replenished indefinitely.<br><br>

A major initiative in our group is to integrate multi-omics datasets generated by state‐of‐the‐art mass spectrometer (MS) and next generation sequencer (NGS) from various stem and progenitor cells and during their differentiation to specialised cell types. The goal is to reconstruct trans-omic networks and to understand how different regulatory machineries (e.g. signalling, transcription, and epigenomics) cooperate to define cell types, functions, and fates, and to translate these knowledge for directed cell type differentiation and tissue/organ regeneration.

<div class="section row">
    <div class="image col-md-5">
        <img class="img-responsive" alt="Project 1 illustration" src="/images/projects/TransReg.png"/ Align="Left">
    </div>
    <div class="col-md-7">
        <h3 class="heading">Reconstructing trans-omic networks in mouse embryonic stem cells (ESCs)</h3>
        <div class="content">
            Pluripotency defines the potential of pluripotent stem cells (PSCs) to differentiate into all somatic and germ cells. We have previously profiled the multi-layered trans‐omics networks in ESCs during their pluripotency transition (<a href="https://doi.org/10.1016/j.cels.2019.03.012">Yang et al. 2019</a>). Building on this work, we are developing methods to characterise signaling cascades, transcriptional networks, and translational protein networks in regulating the transition of pluripotency in ESCs, and to uncover how information is passed on from each of these regulatory layer to the next layer. This project will provide a unique opportunity for developing computational and experimental methods for comprehensive understanding of stem cell systems and their decision-making process.
        </div>
    </div> 

   <div class="col-md-12">     
      <h3 class="heading">'Mini organs' in a dish: identifying molecular controls in generating brain and retina organoids from human PSCs</h3>
      <div class="content">
          While tremendous progress has been made in stem cell therapy, attempts to direct stem/progenitor cells to clinically useful cell types have often been fraught with imprecise and incomplete differentiation. In collaboration with Dr. Anai Gonzalez Cordero's group at CMRI, we are reconstructing trans-networks networks and identifying key network modules and nodes that underlie the generation of brain and retina organoids using human PSCs. Discovery made from these works will have provide both basic and translational knowledge for directed cell type differentiation and tissue/organ regeneration.    
      </div>
   </div>
    
    <div class="col-md-12">
       <h3 class="heading">Identifying molecular controls in muscle cell formation</h3>
       <div class="content">
           In collaboration with Dr. Benjamin Parker's group at the Melbourne University, we are identifying molecular controls that regulate the differentiation of progenitor cells to mature muscle cells. To do this, we have profiled the trans-omic networks on phosphoproteomics and proteomics levels and have developed various computational methods allowing the identification of key phosphorylation events and their downstream transcriptional regulation in controlling myogenesis, formation of muscle cells. Finding from this study will have implications in muscle tissue regeneration.      
      </div>
   </div>    

</div>
<hr>


<h2 class="projects"><b>Interrogating cell identity and cell-fate decisions in spatial contexts and using single-cell multi-omics</b></h2>

The advance of single-cell omics and its recent development towards multimodality, that allows profiling of gene expression, chromatin state, protein abundance and spatial location, creates unprecedented opportunities to study complex biological systems at resolutions that were previously unattainable. Our group is harnessing the power of these new biotechnologies to study how each single cell acquires its identity in organs and tissues and makes decisions during cell differentiation and embryogenesis.

<div class="section row">
    <div class="image col-md-10">
        <img class="img-responsive" alt="Project 2 illustration" src="/images/projects/scMerge.png"/>
    </div>
    <div class="col-md-7">
        <h3 class="heading">Creating a retina cell type altas using single-cell data acquired from primary retina tissues and retina organoids</h3>
        <div class="content">
            We are collaborating with Dr. Anai Gonzalez Cordero's group at CMRI to create the first retina cell type atlas by integrating single-cell omics data generated from various primary retina tissues and retina organoids. From this atlas, we uses our recently developed Cepo algorithm to derive a cell identity score for each gene with respect to the cell types in the atlas (<a href="https://doi.org/10.1038/s43588-021-00172-2">Kim et al. 2021</a>). These cell identity scores provide a basis for annotating cell types and assessing data quality and disease status in future single-cell omics data generated from retina tissues and retina orgnoids.
        </div>
    </div>
    
    <div class="col-md-12">
        <h3 class="heading">Identifying cells that show bipotential differentiation capacity during early embryogenesis</h3>
        <div class="content">
			This project involves experimental profiling of single cells in embryos during their differentiation to multiple cell lineages. Together with Prof. <a href="https://www.cmri.org.au/Research/Research-Units/Embryology/Our-People">Patrick Tam's lab</a>, we are developing experimental and computational techniques for uncovering cells that could differentiate into multiple lineages during early embryogenesis. The results from this study will provide new knowledge to cell lineage specification in embryogensis and shed light on early developmental diseases	
        </div>
    </div>
    
</div>
<hr>

<h2 class="projects"><b>Developing computational methods for pre-processing, analysing, and integrating bulk and single-cell multi-omics data and their spatial information</b></h2>

<div class="section row">
	<div class="col-md-12">
        <h3 class="heading">Project 1: Machine learning application in trans-omics</h3>    
        <div class="content">
			Cell type identification is a key step in single-cell omics experiments. We are in process of developing deep learning-based methods for cell type identification from single-cell multi-omics data.<br><br>
			
			One interesting finding so far is that not all 'similarity' metrics are equally useful in analysing scRNA-seq data (<a href="https://doi.org/10.1093/bib/bby076">Kim et al. 2018</a>).
			Note that most of the clustering and classification methods utilise some kind of similarity metric to compare cells. Therefore, the finding has great implication on both the development
			of computational methods for scNRA-seq data analysis and the biology of individual cells. This project is computational and requires a good grasp of machine learning techniques. Candidates also have
			opportunity to explore deep learning methods and their application in this domain.
        </div>
    </div>

    <div class="col-md-12">
        <h3 class="heading">Project 2: Application of deep learning in bioinformatics</h3>
        <div class="content">
			Deep learning is the latest development in machine learning that has been successfully utilised to address 
			many bioinformatics applications. We are interested in its application such as in transcription factor (TF)
			binding site identification and target gene prediction.<br><br>
			
			The advance in chromatin immunoprecipitation followed by ultrafast sequencing (ChIP-seq) allows the profiling of TF binding sites 
			genome-wide in a population of cells. The massive amount of sequencing data generated from these genome-wide profiling of TF 
			requires sophisticated computational algorithms to be developed for accurately identifying TF binding sites.
			In this project, we aim to develop and apply deep learning models for 
			predicting TF binding sites by integrating ChIP-seq data with other biological knowledge. 
			This project aims to develop and apply cutting-edge deep learning algorithms for solving a key biological problem.
			You will get involved in all aspects of the development including algorithm design, implementation and testing.
        </div>
    </div>
    
    <div class="image col-md-5">
        <img class="img-responsive" alt="Project 4 illustration" src="/images/projects/AdaSampling.png"/>
    </div>
    <div class="col-md-7">
        <h3 class="heading">Project 3: Application of semi-supervised learning in bioinformatics</h3>
        <div class="content">
        
        Identification of transcription factor target genes using machine learning
        
            We have recently developed several semi-supervised and/or knowledge-based learning methods for learning with only positive examples and in the presence of data noise 
            (e.g. <a href="https://doi.org/10.1109/TCYB.2018.2816984">Yang et al. 2018</a>). 
            We are now extending semi-supervised learning methods for various bioinformatics application. This includes the identification of transcription factor target
            genes in differentiating mouse embryonic stem cells (mESCs) by integrating transcriptomics, proteomics, and epigenomics data; the classification of cell types from single-cell
            RNA-seq data.<br><br>
            
            Candidates involved in this project we learn state-of-the-art machine learning methods and their applications to cutting-edge systems biology.
        </div>
    </div>
	
    <div class="col-md-12">
        <h3 class="heading">Project 4: Pathway reconstruction and annotation using statistical models and trans-omics data sets</h3>
        <div class="content">		
			While all cells from a given organism have the same DNA sequence that codes for the same genes, 
			different cell types of that organism only have a subset of genes “turned on”. 
			Genes are commonly annotated into pathways for summarising their collective effect in the biological systems. 
			One of the main drawbacks in current pathway annotation is that they are NOT cell type-specific.<br><br>
			
			We propose to identify and curate cell type-specific pathways, for example, in embryonic stem cells (ESCs) using the
			trans-omics datasets. The key assumption is that genes within a pathway should have correlated expression profile changes when perturbed.
			We have collected ESC differentiation data profiled in a time-course on both proteome and transcriptome levels. 
			Following the above assumption, we aim to (1) identify pathways that are regulated specifically in ESC differentiation; 
			and (2) curate ESC-specific pathways using statistical learning. 
			This project will expose you to the development and application of cutting-edge statistical 
			learning methods to the state-of-the-art bio-molecular applications. It sits at the heart of the interdisciplinary research conducted in our group. 			
        </div>
    </div>
</div>
<hr>

    
<div class="section row">
	<div class="col-md-12">
	<h2 class="heading"><b>Openings</b></h2>
	<ul>
		<li> PhD Scholarships ($39,100 tuition fee plus $25,861 stipend contribution for the duration of PhD) and honours projects are immediately available. Successful candidates will undertake 
		exciting new projects in the general areas of Systems and Computational Biology.</li>
		<li> Summer Research Scholarships ($513 per week for 8 weeks) are available for third year undergraduates who are interested
		in exploring either 'wet' or 'dry' research projects in computational systems biology and consider pursuing an honours project in our group.</li>
	</ul>
	
	For postgraduate applicants, all 'wet' projects require experience with cell culturing and good understanding of basic experimental protocols (e.g. qPCR, westernblotting) and preferably experience of (single cell-)sequencing and
	mass spectrometry-based proteomics. All 'dry' projects require programming skill
	in at least one programming/scripting language (R, Perl, Python, Java, C++, C and Matlab). There are ample opportunity to
	work closely with biologists, statisticians, bioinformaticians, and computer scientists.
	<br><br>
	To Apply: Please send CV via email to Dr. Pengyi Yang (<a href="mailto:pengyi.yang@sydney.edu.au">pengyi.yang [at] sydney.edu.au</a>)
	<br><br>
	More information about scholarship opportunity at the University of Sydney can be found <a href="https://sydney.edu.au/study/study-options/postgraduate-research.html">here</a>.<br>
	For obtaining scholarships and top-ups from CMRI, please read <a href="https://www.cmri.org.au/Research/For-Students">here</a>.<br><br>
	
	Below is an <b>incomplete</b> list of research themes and potential/ongoing projects in our lab. Candidates are welcome to get in touch and discuss these 
	and/or any other potential projects aligned with one or more of our research themes.<br><br>
	
	</div>
</div>    

</div>
















