---
layout: default
---

<div class="container">
<h2 class="projects">On going projects</h2>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 1 illustration" src="/images/projects/ESC.png"/ Align="Left">
    </div>
    <div class="col-md-8">
        <h3 class="heading">Mapping and modelling trans-omic networks: a machine learning and omics integration approach</h3>
        <div class="content">
            This is a major initiative in our group to integrate trans-omics dataset generated from mouse embryonic stem cells
            (mESCs) differentiation process. The goal is to investigate the cross-talk of signalling cascades, epigenomic,
            transcriptomic and proteomic regulation, and there feedback regulations.
            The high-throughput data that are are seeking to integrate include time-course mass spectrometry-based proteomics and phosphoproteomics, RNA-seq,
            and ChIP-seq data.
            <br>
            People: Pengyi Yang, Sean Humphrey, Dinuka Perera
        </div>
    </div>
</div>

<div class="section row">
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 2 illustration" src="/images/projects/MixtureModel.png"/>
    </div>
    <div class="col-md-8">
        <h3 class="heading">Mixture modelling in single-cell RNA-seq data</h3>
        <div class="content">
            We are working closely with Prof. Jean Yang's group on mixture modelling from single-cell RNA-seq (scRNA-seq) data. This include
            developing novel statistical model to capture various unique aspects in scRNA-seq data. We are applying our model
            to understand cell differentiation and tissue development processes in human and mouse.
            <br>
            Peaple: Taiyun Kim, Shila Ghazanfar, Jean Yang
            <br>
            Relevant publications: <a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-016-0370-4">here</a>
        </div>
    </div>
</div>
<div class="section row">
    <div class="col-md-8">
        <h3 class="heading">Identification of transcription factor target genes using machine learning</h3>
        <div class="content">
            We have recently developed an adaptive sampling approach for learning from positive-unlabeled dataset. 
            We are extending this semi-supervised learning approach for the identification of transcription factor target
            genes in differentiating mouse embryonic stem cells (mESCs) by integrating transcriptomic and epigenomic data.             
            <br>
            People: Dinuka Perera, Chendong Ma
            <br>
            Relevant publications: <a href="http://www.ijcai.org/proceedings/2017/0457.pdf">here</a>
        </div>
    </div>
    <div class="image col-md-4">
        <img class="img-responsive" alt="Project 4 illustration" src="/images/projects/AdaSampling.png"/>
    </div>
</div>
<br><br>

<hr>
<h2 class="projects">Potential new projects</h2>
 
<div class="section row">
    <div class="col-md-8">
        <h3 class="heading">Understand the formation of fat tissue</h3>
        <div class="content">
            The understanding of fat tissue formation is critical for treating various metabolic and obesity related diseases.
            In collaboration with Prof. David James lab, we have developed several machine learning approach to elucidate the
            signaling cascades in fat cells (Humphrey et al. 2013 Cell Metab.). We are planning for global profiling of the 
            fat cell differentiation process to understand the trans-omic networks underlying this process 
            using various statistical modelling approaches.
            <br>
            People: Pengyi Yang, Sean Humphrey, David James
            <br>
            Relevant publications: <a href="http://www.ijcai.org/proceedings/2017/0457.pdf">here</a>
        </div>
    </div>
</div>

</div>
