---
layout: default
---

<div class="container">
<h2 class="projects">Theme: mapping and modelling trans-regulatory networks</h2>

<div class="section row">
    <div class="image col-md-3">
        <img class="img-responsive" alt="Project 1 illustration" src="/images/projects/TransReg.png"/ Align="Left">
    </div>
    <div class="col-md-9">
        <h3 class="heading">Reconstructing trans-regulatory networks by trans-omic data integration</h3>
        <div class="content">
            A major initiative in our group is to integrate trans-omics datasets generated by state‐of‐the‐art mass spectrometer (MS) and next 
			generation sequencer (NGS) from various cell systems.
            The goal is to reconstruct trans-regulatory networks and to understand how different regulatory machineries (e.g. signalling, transcription, and epigenomics) 
            co-operate to define cell states, functions, and fates.<br><br>
            
            We have previously developed various computational methods to integrate the multi-layered trans‐omics datasets generated during naive to formative pluripotency transition 
            in ESCs (<a href="https://doi.org/10.1016/j.cels.2019.03.012">Yang et al. (2019)</a>). Our current research project 
			aims to further this study by developing methods to characterise signaling cascades, transcriptional networks, and translational protein networks and their cross‐talks. 
			This project will provide a unique opportunity for developing computational methods for comprehensive understanding of cell systems and their decision-making process.
        </div>
    </div> 
        <hr>
    
    <div class="col-md-12">
        <h3 class="heading">Understanding the formation of fat tissue</h3>
        <div class="content">
            The understanding of fat tissue formation is critical for treating various metabolic and obesity related diseases.
            In collaboration with Prof. David James lab, we have developed several machine learning approaches to elucidate the
            signaling cascades in fat cells (<a href="https://doi.org/10.1093/bioinformatics/btv550">Yang et al. (2016)</a>; <a href="https://doi.org/10.1371/journal.pcbi.1004403">Yang et al. (2015)</a>;
			<a href="https://dx.doi.org/10.1016%2Fj.cmet.2013.04.010">Humphrey et al. (2013)</a>). We are planning for global profiling of the 
            fat cell (3T3-L1) differentiation process to understand the trans-omic networks underlying this process 
            using various statistical modelling approaches. You will get involved in developing key statistical models to
            integrate trans-omics data and infer their feedback regulation during the maturation process of fat cells (i.e. adipogenesis).
        </div>
        <hr>
   </div>     

   <div class="col-md-12">     
        ## stem cell therupy 
        
    </div>
</div>
<hr>


<h2 class="projects">Theme: single-cell biology and omics</h2>

<div class="section row">
    <div class="image col-md-3">
        <img class="img-responsive" alt="Project 2 illustration" src="/images/projects/MixtureModel.png"/>
    </div>
    <div class="col-md-9">
        <h3 class="heading">Multiple single-cell RNA-seq data integration</h3>
        <div class="content">
            We are working closely with Prof. Jean Yang's group on mixture modelling from single-cell RNA-seq (scRNA-seq) data. This include
            developing novel statistical models to capture various unique aspects in scRNA-seq data. We are applying our model
            to understand cell differentiation and tissue development processes in human and mouse.
        </div>
    </div>
    <hr>
    
    <div class="col-md-9">
        <h3 class="heading">Cell type identification using unsupervised, semi-supervised and supervised learning</h3>
        <div class="content">

        </div>
    </div>
</div>
<hr>


<h2 class="projects">Potential new projects</h2>

<hr>
<h2 class="projects">Theme: computational methodology innovation in bioinformatics</h2>

<div class="section row">
	<div class="col-md-12">
        <h3 class="heading">Machine learning application in trans-omics</h3>
        <div class="content">
			Build on top of mapping and modelling trans-omic networks, A PhD position is available in developing and applying machine 
			learning models to multi-layered trans‐omics datasets generated by MS-based phospho(proteomics) and NGS-based sequencing.<br><br>

			Our research project, funded by the Australian Research Council (ARC), aims to develop novel machine learning algorithms 
			to analyse and integrate large‐scale MS‐based omics data with ultra‐fast NGS‐based omics data generated from complex 
			biological systems. Characterising the signaling cascades, transcriptional networks, and translational protein networks 
			and their cross‐talks are critical for comprehensive understanding of complex biological systems. Our large‐scale 
			multi-layered trans‐omics data generated from state‐of‐the‐art technological platforms provide a unique opportunity to 
			uncover novel biology and molecular mechanisms that are critical for treating complex diseases and personalised medicine. 
    	</div>
    </div>
	<hr>

    <div class="col-md-12">
        <h3 class="heading">Application of deep learning in bioinformatics</h3>
        <div class="content">
			Deep learning is the latest development in machine learning that has been successfully utilised to address 
			many bioinformatics applications. We are interested in its application such as in transcription factor (TF)
			binding site identification and target gene prediction.<br><br>
			
			The advance in chromatin immunoprecipitation followed by ultrafast sequencing (ChIP-seq) allows the profiling of TF binding sites 
			genome-wide in a population of cells. The massive amount of sequencing data generated from these genome-wide profiling of TF 
			requires sophisticated computational algorithms to be developed for accurately identifying TF binding sites.
			In this project, we aim to develop and apply deep learning models for 
			predicting TF binding sites by integrating ChIP-seq data with other biological knowledge. 
			This project aims to develop and apply cutting-edge deep learning algorithms for solving a key biological problem.
			You will get involved in all aspects of the development including algorithm design, implementation and testing.
        </div>
    </div>
    <hr>
    
    <div class="image col-md-3">
        <img class="img-responsive" alt="Project 4 illustration" src="/images/projects/AdaSampling.png"/>
    </div>
    <div class="col-md-9">
        <h3 class="heading">Identification of transcription factor target genes using machine learning</h3>
        <div class="content">
            We have recently developed an adaptive sampling approach for learning from positive-unlabeled dataset. 
            We are extending this semi-supervised learning approach for the identification of transcription factor target
            genes in differentiating mouse embryonic stem cells (mESCs) by integrating transcriptomics, proteomics, and epigenomics data.             
            <br>
            People: Dinuka Perera, Chendong Ma, Pengyi Yang
            <br>
            Relevant publications: <a href="http://www.ijcai.org/proceedings/2017/0457.pdf">here</a>
        </div>
    </div>
	<hr>
	
    <div class="col-md-12">
        <h3 class="heading">Pathway reconstruction and annotation using statistical models and trans-omics data sets</h3>
        <div class="content">		
			While all cells from a given organism have the same DNA sequence that codes for the same genes, 
			different cell types of that organism only have a subset of genes “turned on”. 
			Genes are commonly annotated into pathways for summarising their collective effect in the biological systems. 
			One of the main drawbacks in current pathway annotation is that they are NOT cell type-specific.<br><br>
			
			We propose to identify and curate cell type-specific pathways, for example, in embryonic stem cells (ESCs) using the
			trans-omics datasets. The key assumption is that genes within a pathway should have correlated expression profile changes when perturbed.
			We have collected ESC differentiation data profiled in a time-course on both proteome and transcriptome levels. 
			Following the above assumption, we aim to (1) identify pathways that are regulated specifically in ESC differentiation; 
			and (2) curate ESC-specific pathways using statistical learning. 
			This project will expose you to the development and application of cutting-edge statistical 
			learning methods to the state-of-the-art bio-molecular applications. It sits at the heart of the interdisciplinary research conducted in our group. 			
        </div>
    </div>
</div>


<hr>
<div class="section row">
	<div class="col-md-12">
	<h2 class="heading"><b>Openings</b></h2>
	PhD scholarships ($39,100 tuition fee plus $25,861 stipend contribution for the duration of PhD) and honours projects are immediately available. Successful candidates will undertake 
	exciting new projects in the general areas of Systems and Computational Biology. 
	
	All 'wet' projects require experience with cell culturing and good understanding of basic experimental protocols (e.g. qPCR, westernblotting) and preferably experience of (single cell)-sequencing and
	mass spectrometry-based proteomics. All 'dry' projects require programming skill
	in at least one programming/scripting language (R, Perl, Python, Java, C++, C and Matlab). There are ample opportunity to
	work closely with biologists, statisticians, bioinformaticians, and computer scientists.
	<br><br>
	To Apply: Please send CV via email to Dr. Pengyi Yang
	<br><br>
	More information about scholarship opportunity at the University of Sydney can be found <a href="http://sydney.edu.au/study/find-a-course/postgraduate-study.html">here</a>.<br>
	For obtaining scholarships and top-ups from CMRI, please read <a href="https://www.cmri.org.au/Research/For-Students">here</a>.
	</div>
</div>

</div>
