---
layout: default
---

<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>


<div class="container">
      <h2>Notations:</h2>
      <br>
            <b>Bold</b>: CSB group member<br>
    		&#10018;: Co-first author<br>
    		&dagger;: Corresponding/Co-corresponding author<br>
    
      <hr>

	<h2 class="publications">Key publications</h2>
	<br>

        <! -- Matilda -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/Matilda.jpg"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Multi-task learning from multimodal single-cell omics with Matilda</b>
                <br><br>
                <b>Liu, C.</b>, <b>Huang, H.</b> & <b><u>Yang, P.</u></b><sup>&dagger;</sup>

                <br><br>
                <b><i>Nucleic Acids Research</i></b>, gkad157 (2023). [<a href="https://doi.org/10.1093/nar/gkad157">Full Text</a>], [<a href="https://github.com/PYangLab/Matilda">Repo and tutorial</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/nar/gkad157"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1093/nar/gkad157"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>    
    
        <! -- Retinal Organoids -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/retinalOrganoids.jpg"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Comprehensive characterization of fetal and mature retinal cell identity to assess the fidelity of retinal organoids</b>
                <br><br>
                <b>Kim, H.</b>, O'Hara-Wright, M., <b>Kim, D.</b>, Loi, T., Lim, B., Jamieson, R., Gonzalez-Cordero, A.<sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup>
                <br><br>
                <b><i>Stem Cell Reports</i></b>, 18(1), 175-189 (2023). [<a href="https://doi.org/10.1016/j.stemcr.2022.12.002">Full Text</a>], [<a href="https://shiny.maths.usyd.edu.au/Eikon/">Eikon Shiny Web Server</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/2022.06.13.495996"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1016/j.stemcr.2022.12.002"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>
    
        <! -- Cepo -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/Cepo2021.png"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Uncovering cell identity through differential stability with Cepo</b>
                <br><br>
                <b>Kim, H.</b>, Wang, K., <b>Chen, C.</b>, Lin, Y., Tam, PPL., Lin, D., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup>
                <br><br>
                <b><i>Nature Computational Science</i></b>, 1, 784-790 (2021). [<a href="https://doi.org/10.1038/s43588-021-00172-2">Full Text</a>], [<a href="https://rdcu.be/cDyXi">Nature Content Sharing link</a>], [<a href="https://bioconductor.org/packages/release/bioc/html/Cepo.html">BioC R package</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s43588-021-00172-2"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1038/s43588-021-00172-2"></span>
                    </div>
                </div>
            </div>
        </div>
        <br> 
    
        <! -- Number of cluster estimation -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/estNumClust.png"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data</b>
                <br><br>
                <b>Yu, L.</b>, Cao, Y., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup>
                <br><br>
                <b><i>Genome Biology</i></b>, 23, 49 (2022). [<a href="https://doi.org/10.1186/s13059-022-02622-0">Full Text</a>], [<a href="https://github.com/PYangLab/scCCESS">Repo</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1186/s13059-022-02622-0"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1186/s13059-022-02622-0"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>
    
        <! -- PhosR -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/PhosR.jpg"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>PhosR enables processing and functional analysis of phosphoproteomic data</b>
                <br><br>
                <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup>
                <br><br>
                <b><i>Cell Reports</i></b>, 34(8), 108771 (2021). [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>], [<a href="https://pyanglab.github.io/PhosR/">BioC R package</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.celrep.2021.108771"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1016/j.celrep.2021.108771"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>
    
        <! -- NAR pluripotency -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/NARpluripotency.png"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Transcriptional network dynamics during the progression of pluripotency revealed by integrative statistical learning</b>
                <br><br>
                <b>Kim, H.</b>, Osteil, P., Humphrey, S., Cinghu, S., Oldfield, A., Patrick, E., Wilkie, E., Peng, G., Suo, S., Jothi, R., Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup>
                <br><br>
                <b><i>Nucleic Acids Research</i></b>, 48(4), 1828-1842 (2020). [<a href="https://doi.org/10.1093/nar/gkz1179">Full Text</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1093/nar/gkz1179"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1093/nar/gkz1179"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>        
    
        <! -- Ensemble deep learning -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/EnsembleDeepLearning.png"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Ensemble deep learning in bioinformatics</b>
                <br><br>
                Cao, Y., <b>Geddes, T.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup>
                <br><br>
                <b><i>Nature Machine Intelligence</i></b>, 2, 500-508 (2020). [<a href="https://doi.org/10.1038/s42256-020-0217-y">Full Text</a>], [<a href="https://rdcu.be/b6jIn">Nature Content Sharing link</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s42256-020-0217-y"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1038/s42256-020-0217-y"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>    

        <! -- scClassify -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/scClassify.png"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>scClassify: sample size estimation and multiscale classification of cells using single and multiple reference</b>
                <br><br>
                Lin, Y., Cao, Y., <b>Kim, H.</b>, Salim, A., Speed, T., Lin, D., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup>
                <br><br>
                <b><i>Molecular Systems Biology</i></b>, 16, e9389 (2020). [<a href="https://doi.org/10.15252/msb.20199389">Full Text</a>], [<a href="http://www.bioconductor.org/packages/release/bioc/html/scClassify.html">BioC R package</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.15252/msb.20199389"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.15252/msb.20199389"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>    
    
        <! -- Pluripotency -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/Pluripotency2019.jpg"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Multi-omic profiling reveals dynamics of the phased progression of pluripotency</b>
                <br><br>
                <b><u>Yang, P.</b></u><sup>&#10018;</sup><sup>&dagger;</sup>, Humphrey, S.<sup>&#10018;</sup><sup>&dagger;</sup>, Cinghu, S.<sup>&#10018;</sup>, Pathania, R., Oldfield, A., Kumar, D., <b>Perera, D.</b>, Yang, J., James, D., Mann, M. & Jothi, R.<sup>&dagger;</sup>
                <br><br>
                <b><i>Cell Systems</i></b>, 8(5), 427-445 (2019). [<a href="https://doi.org/10.1016/j.cels.2019.03.012">Full Text</a>], [<a href="http://stemcellatlas.org/">The Stem Cell Atlas</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.cels.2019.03.012"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1016/j.cels.2019.03.012"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>
    
        <! -- scMerge -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/scMerge.png"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets</b>
                <br><br>
                Lin, Y., Ghazanfar, S., Wang, K., Gagnon-Bartsch, J., Lo, K., Su, X., Han, Z., Ormerod, J., Speed, T., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup>
                <br><br>
                <b><i>Proceedings of the National Academy of Sciences of the United States of America</i></b>, 116(20), 9775-9784 (2019). [<a href="https://doi.org/10.1073/pnas.1820006116">Full Text</a>], [<a href="https://bioconductor.org/packages/release/bioc/html/scMerge.html">BioC R package</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1073/pnas.1820006116"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1073/pnas.1820006116"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>
    
        <! -- Attenuator -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/Attenuator2017.jpg"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Intragenic enhancers attenuate host gene expression</b>
                <br><br>
                Cinghu, S.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Kosak, J., Conway, A., Kumar, D., Oldfield, A., Adelman, K. & Jothi, R.
                <br><br>
                <b><i>Molecular Cell</i></b>, 68(1), 104–117 (2017). [<a href="https://doi.org/10.1016/j.molcel.2017.09.010">Full Text</a>], [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Attenuator.pdf">PDF</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.molcel.2017.09.010"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1016/j.molcel.2017.09.010"></span>
                    </div>
                </div>
            </div>
        </div>
        <br>
    
        <! -- NFY -->
        <div class="section row" style="background-color: #F9F9F9;">
            <div class="image col-md-3">
                <img class="img-responsive" src="/Publications/image/NFY2014.jpg"/ Align="Left">
            </div>

            <div class="col-md-9">
                <br>
                <b>Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors</b>
                <br><br>
                Oldfield, A.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Conway, A., Cinghu, S., Freudenberg, J., Yellaboina, S. & Jothi, R.
                <br><br>
                <b><i>Molecular Cell</i></b>, 55(5), 708-722 (2014). [<a href="https://doi.org/10.1016/j.molcel.2014.07.005">Full Text</a>], [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/NFY.pdf">PDF</a>]
                <br><br>
                <div style="display: flex;">
                    <div style="margin-right: 10px;">
                        <span class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.molcel.2014.07.005"></span>
                    </div>
                    <div style="margin-right: 10px;">
                        <span class="__dimensions_badge_embed__" data-style="small_circle" data-doi="10.1016/j.molcel.2014.07.005"></span>
                    </div>
                </div>
            </div>
        </div>
        <br><br>
    
	<hr>
		
	
    <h2 class="publications">All publications</h2>
    <!--<h4> You can find a full publication list at <a href=https://scholar.google.com.au/citations?user=s-cliFIAAAAJ&hl=en target="_blank">Google Scholar.</a></h4>-->
    <br>
		
        <h3>2023</h3>
            <font color="#8B0000"><b>Systems biology:</b></font><br>
            96. <b>Kim, H.</b>, O'Hara-Wright, M., <b>Kim, D.</b>, Loi, T., Lim, B., Jamieson, R., Gonzalez-Cordero, A.<sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2023)
            Comprehensive characterization of fetal and mature retinal cell identity to assess the fidelity of retinal organoids.
            <b><i>Stem Cell Reports</i></b>, 18(1), 175-189.
            [<a href="https://doi.org/10.1016/j.stemcr.2022.12.002">Full Text</a>]
            [<a href="https://shiny.maths.usyd.edu.au/Eikon/">Eikon Shiny Web Server</a>]
            <br><br>
    
            95. Fazakerley, D., Gerwen, J., Cooke, K., Duan, X., Needham, E., Díaz-Vegas, A., Madsen, S., Norris, D., Shun-Shion, A., Krycer, J., Burchfield, J., <b><u>Yang, P.</u></b>, Wade, M., Brozinick, J., James, D. & Humphrey, S. (2023)
            Phosphoproteomics reveals rewiring of the insulin signaling network and multi-nodal defects in insulin resistance.
            <b><i>Nature Communications</i></b>, 14, 923.
            [<a href="https://doi.org/10.1038/s41467-023-36549-2">Full Text</a>]
            <br><br>
    
            <font color="#8B0000"><b>Methodology and tools:</b></font><br>
            94. <b>Liu, C.</b>, <b>Huang, H.</b> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2023)
            Multi-task learning from multimodal single-cell omics with Matilda.
            <b><i>Nucleic Acids Research</i></b>, gkad157.
            [<a href="https://doi.org/10.1093/nar/gkad157">Full Text</a>]
            [<a href="https://github.com/PYangLab/Matilda">Repo and tutorial</a>]
            <br><br>
    
            93. Patrick, E., Canete, N., Iyengar, S., Harman, A., Sutherland, G. & <b><u>Yang, P.</u></b> (2023)
            Spatial analysis for highly multiplexed imaging data to identify tissue microenvironments.
            <b><i>Cytometry Part A</i></b>, DOI: 10.1002/cyto.a.24729.
            [<a href="https://doi.org/10.1002/cyto.a.24729">Full Text</a>]
            <br><br>
    
            92. <b>Kim, T.</b>, <b>Kim, H.</b>, Oldfield, A. & <b><u>Yang, P.</u></b> (2023)
            PAD2: interactive exploration of transcription factor genomic colocalization using ChIP-seq data.
            <b><i>STAR Protocols</i></b>, 4(2), 102203.
            [<a href="https://doi.org/10.1016/j.xpro.2023.102203">Full Text</a>]
            [<a href="http://pad2.maths.usyd.edu.au/">PAD2 Web Server</a>]
            <br><br>
    
            91. Cao, Y., Tran, A., <b>Kim, H.</b>, Robertson, N., Lin, Y., Torkel, M., <b><u>Yang, P.</u></b>, Patrick, E., Ghazanfar, S. & Yang, J. (2023)
            Thinking process templates for constructing data stories with SCDNEY.
            <b><i>F1000Research</i></b>, 12, 261.
            [<a href="https://doi.org/10.12688/f1000research.130623.1">Full Text</a>]
            <br><br>
    
        <h3>2022</h3>
            <font color="#8B0000"><b>Systems biology:</b></font><br>
            90. <b>Kim, H.</b>, Li, P., <b>Kim, T.</b>, Oldfield, A., Zheng, X. <sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2022)
            Integrative Analysis Reveals Histone Demethylase LSD1 Promotes RNA Polymerase II Pausing.
            <b><i>iScience</i></b>, 25(10), 105049.
            [<a href="https://doi.org/10.1016/j.isci.2022.105049">Full Text</a>]
            <br><br>
    
            89.
            Blazev, R., Carl, C., Ng, Y., Molendijk, J., Voldstedlund, C., Zhao, Y., <b>Xiao, D.</b>, Kueh, A., Miotto, P., Haynes, V., Hardee, J., Chung, J., McNamara, J., Qian, H., Gregorevic, P., Oakhill, J., Herold, M., Jensen, T., Lisowski, L., Lynch, G., Dodd, G., Watt, M., <b><u>Yang, P.</b></u>, Kiens, B., Richter, E., Parker, B. (2022) 
            Phosphoproteomics of three exercise modalities identifies canonical signaling and C18ORF25 as an AMPK substrate regulating skeletal muscle function.
            <b><i>Cell Metabolism</i></b>, 34(10), 1561-1577.
            [<a href="https://doi.org/10.1016/j.cmet.2022.07.003">Full Text</a>]
            <br><br>

            88.
            <b>Xiao, D.</b>, Caldow, M., <b>Kim, H.</b>, Blazev, R., Koopman, R., Manandi, D., Parker, B. <sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2022)
            Time-resolved phosphoproteome and proteome analysis reveals kinase signalling on master transcription factors during myogenesis.
            <b><i>iScience</i></b>, 25(6), 104489.
            [<a href="https://doi.org/10.1016/j.isci.2022.104489">Full Text</a>]
            <br><br>

            87.
            <b><u>Yang, P.</u></b> & Tam, P. (2022)
            Mouse organogenesis atlas at single-cell resolution.
            <b><i>Cell</i></b>, 185(10), 1625-1627.
            [<a href="https://doi.org/10.1016/j.cell.2022.04.026">Full Text</a>]
            <br><br>
    
            86.
            Fernando, M., Lee, S., Wark, J., <b>Xiao, D.</b>, Lim, B., O'Hara-Wright, M., <b>Kim, H.</b>, Smith, G., Wong, T., Teber, E., Ali, R., <b><u>Yang, P.</u></b>, Graham, M. & Gonzalez-Cordero, A. (2022)
            Differentiation of brain and retinal organoids from confluent cultures of pluripotent stem cells connected by nerve-like axonal projections of optic origin.
            <b><i>Stem Cell Reports</i></b>, 17(6), 1476-1492.
            [<a href="https://doi.org/doi:10.1016/j.stemcr.2022.04.003">Full Text</a>]
            <br><br>
    
            85.
            <b>Xiao, D.</b>, <b>Chen, C.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2022)
            Computational systems approach towards phosphoproteomics and their downstream regulation.
            <b><i>Proteomics</i></b>, e2200068.
            [<a href="https://doi.org/10.1002/pmic.202200068">Full Text</a>]
            [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Computational systems approach towards phosphoproteomics and their downstream regulation.pdf">PDF</a>]
            <br><br>
            
            84.
            <b>Xiao, D.</b>, <b>Kim, H.</b>, Pang, I. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2022)
            Functional analysis of the stable phosphoproteome reveals cancer vulnerabilities.
            <b><i>Bioinformatics</i></b>, 38(7), 1956-1963.
            [<a href="https://doi.org/10.1093/bioinformatics/btac015">Full Text</a>]
            [<a href="/resources/Supplementary Table 2.xlsx">List of stably phosphorylated sites</a>]
            <br><br>

            83.
            Duan, X., Norris, D., Humphrey, S., <b><u>Yang, P.</u></b>, Cooke, K., Bultitude, W., Parker, B., Conway, O., Burchfield, J., Krycer, J., Brodsky, F., James, D. & Fazakerley, D. (2022)
            Trafficking regulator of GLUT4-1 (TRARG1) is a GSK3 substrate
            <b><i>Biochemical Journal</i></b>, 479(11), 1237-1256.
            [<a href="https://doi.org/10.1042/BCJ20220153">Full Text</a>]
            <br><br>    
    
            82.
            Tu, J., Li, W., Yang, S., <b><u>Yang, P.</u></b>, Yan, Q., Wang, S., Lai, K., Bai, X., Wu, C., Ding, W., Cooper‐White, J., Diwan, A., Yang, C., Yang, H. & Zou, J. (2022)
            Single‐cell transcriptome profiling reveals multicellular ecosystem of nucleus pulposus during degeneration progression.
            <b><i>Advanced Science</i></b>, 9(3), 2103631.
            [<a href="https://doi.org/10.1002/advs.202103631">Full Text</a>]
            <br><br>
    
            <font color="#8B0000"><b>Methodology and tools:</b></font><br>
            81.
            <b>Yu, L.</b>, Cao, Y., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2022)
            Benchmarking clustering algorithms on estimating the number of cell types from single-cell RNA-sequencing data.
            <b><i>Genome Biology</i></b>, 23, 49.
            [<a href="https://doi.org/10.1186/s13059-022-02622-0">Full Text</a>]
            [<a href="https://github.com/PYangLab/scCCESS">Repo</a>]
            <br><br>
    
            80.
            Cao, Y., Lin, Y., Patrick, E., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2022)
            scFeatures: Multi-view representations of single-cell and spatial data for disease outcome prediction.
            <b><i>Bioinformatics</i></b>, 38(20), 4745-4753.
            <br><br>
    
            79. 
            Tran, A., <b><u>Yang, P.</b></u>, Yang, J. & Ormerod, J. (2022)
            Computational approaches for direct cell reprogramming: from the bulk omics era to the single cell era.
            <b><i>Briefings in Functional Genomics</i></b>, 21(4), 270-279.
            [<a href="https://doi.org/10.1093/bfgp/elac008">Full Text</a>]
            <br><br>
    
            78. 
            Tran, A., <b><u>Yang, P.</b></u>, Yang, J. & Ormerod, J. (2022)
            scREMOTE: Using multimodal single cell data to predict regulatory gene relationships and to build a computational cell reprogramming model.
            <b><i>NAR Genomics and Bioinformatics</i></b>, 4(1), lqac023.
            [<a href="https://doi.org/10.1093/nargab/lqac023">Full Text</a>]
            <br><br>
    
        <h3>2021</h3>
            <font color="#8B0000"><b>Systems biology:</b></font><br>
            77.
            Kumar, D., Cinghu, S., Oldfield, A., <b><u>Yang, P.</b></u> & Jothi, R. (2021)
            Decoding the function of bivalent chromatin in development and cancer.
            <b><i>Genome Research</i></b>, 31(12), 2170-2184.
            [<a href="https://doi.org/10.1101/gr.275736.121">Full Text</a>]
            <br><br>
    
            76.
            Kearney, A., Norris, D., Ghomlaghi, M., Wong, M., Humphrey, S., Carroll, L., Yang, G., Cooke, K., <b><u>Yang, P.</b></u>, <b>Geddes, T.</b>, Shin, S., Fazakerley, D., Nguyen, L., James, D. & Burchfield, J. (2021)
            Akt phosphorylates insulin receptor substrate to limit PI3K-mediated PIP3 synthesis.
            <b><i>eLife</i></b>, 10, e66942.
            [<a href="https://doi.org/10.7554/eLife.66942">Full Text</a>]
			<br><br>
    
            75.
            <b>Kim, H.</b>, Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Defining cell identity beyond the premise of differential gene expression.
            <b><i>Cell Regeneration</i></b>, 10, 20.
            [<a href="https://doi.org/10.1186/s13619-021-00083-7">Full Text</a>]
			<br><br>
    
            74.
            Recasens, A., Humphrey, S., Ellis, M., Hoque, M., Abbassi, R., Chen, B., Longworth, M., Needham, E., James, D., Johns, T., Day, B., Kassiou, M., <b><u>Yang, P.</b></u> & Munoz, L. (2021)
            Global phosphoproteomics reveals DYRK1A regulates CDK1 activity in glioblastoma cells.
            <b><i>Cell Death Discovery</i></b>, 7, 81.
            [<a href="https://doi.org/10.1038/s41420-021-00456-6">Full Text</a>]
            <br><br>
    
            73.
            Norris, D.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Shin, S.<sup>&#10018;</sup>, Kearney, A., <b>Kim, H.</b>, <b>Geddes, T.</b>, Senior A., Fazakerley, D., Nguyen, L., James, D. & Burchfield, J. (2021)
            Signalling heterogeneity is defined by pathway architecture and inter-cellular variability in protein expression.
            <b><i>iScience</i></b>, 24(2), 102118.
            [<a href="https://doi.org/10.1016/j.isci.2021.102118">Full Text</a>]
            <br><br>
    
            72.
            Singh, R., Cottle, L., Loudovaris, T., <b>Xiao, D.</b>, <b><u>Yang, P.</u></b>, Thomas, H., Kebede M. & Thorn P. (2021)
            Enhanced structure and function of human pluripotent stemcell-derived beta-cells cultured on extracellular matrix.
            <b><i>STEM CELLS Translational Medicine</i></b>, 10, 492-505.
            [<a href="https://doi.org/10.1002/sctm.20-0224">Full Text</a>]
            <br><br>
    
            71.
            Su, X., Zhao, L., Shi, Y., Zhang, R., Long, Q., Bai, S., Luo, Q., Lin, Y., Zou, X., Ghazanfar, S., Tao, K., Yang, G., Wang, L., He, K., Cui, X., He, J., Wu, J., Han, B., Yan, B., Deng, B., Wang, N., Li, X., <b><u>Yang, P.</u></b>, Hou, S., Sun, J., Yang, J., Chen, J. & Han, Z. (2021)
            Clonal evolution in liver cancer at single-cell and single-variant resolution.
            <b><i>Journal of Hematology & Oncology</i></b>, 14, 22.
            [<a href="https://doi.org/10.1186/s13045-021-01036-y">Full Text</a>]
            <br><br>
    
            70.
            Blazev, R., Ashwood, C., Abrahams, J., Chung, L., Francis, D., <b><u>Yang, P.</u></b>, Watt, K., Qian, H., Quaife-Ryan, G., Hudson, J., Gregorevic, P., Thaysen-Andersen, M. & Parker, B. (2021)
            Integrated glycoproteomics identifies a role of N-glycosylation and galectin-1 on myogenesis and muscle development.
            <b><i>Molecular & Cellular Proteomics</i></b>, 20, 100030.
            [<a href="https://doi.org/10.1074/mcp.RA120.002166">Full Text</a>]
            <br><br>
    
		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
            
            69.
            <b>Kim, H.</b>, Wang, K., <b>Chen, C.</b>, Lin, Y., Tam, PPL., Lin, D., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Uncovering cell identity through differential stability with Cepo.
            <b><i>Nature Computational Science</i></b>, 1, 784-790.
            [<a href="https://doi.org/10.1038/s43588-021-00172-2">Full Text</a>]
            [<a href="https://rdcu.be/cDyXi">Nature Content Sharing link</a>]
            [<a href="https://bioconductor.org/packages/release/bioc/html/Cepo.html">BioC R package</a>]
            <br><br>
    
            68.
            <b><u>Yang, P.</b></u><sup>&dagger;</sup>, <b>Hao, H.</b> & <b>Liu, C.</b> (2021)
            Feature selection revisited in the single-cell era.
            <b><i>Genome Biology</i></b>, 22, 321.
            [<a href="https://doi.org/10.1186/s13059-021-02544-3">Full Text</a>]
            <br><br>
    
            67.
            Cao, Y., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2021)
            A benchmark study of simulation methods for single-cell RNA sequencing data.
            <b><i>Nature Communications</i></b>, 12, 6911.
            [<a href="https://doi.org/10.1038/s41467-021-27130-w">Full Text</a>]
            [<a href="https://github.com/SydneyBioX/SimBench">Repo</a>]
            [<a href="http://shiny.maths.usyd.edu.au/">Shiny</a>]
            <br><br>
    
            66.
            <b>Kim, H.</b>, <b>Kim, T.</b>, <b>Xiao, D.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR.
            <b><i>STAR Protocols</i></b>, 2(2), 100585.
            [<a href="https://doi.org/10.1016/j.xpro.2021.100585">Full Text</a>]
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
    
            65.
            <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            PhosR enables processing and functional analysis of phosphoproteomic data.
            <b><i>Cell Reports</i></b>, 34(8), 108771.
            [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
            [<a href="https://pyanglab.github.io/PhosR/">BioC R package</a>]
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
    
            64.
            <b>Kim, T.</b>, Tang, O., Vernon, S., Kott, K., Koay, Y., Park, J., James, D., Grieve, S., Speed, T., <b><u>Yang, P.</u></b><sup>&dagger;</sup>, Figtree, G.<sup>&dagger;</sup>, O'Sullivan, J.<sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2021)
            A hierarchical approach to removal of unwanted variation for large-scale metabolomics data.
            <b><i>Nature Communications</i></b>, 12, 4992.
            [<a href="https://doi.org/10.1038/s41467-021-25210-5">Full Text</a>]
            [<a href="https://github.com/SydneyBioX/hRUV">Repo</a>]
            <br><br>    
    
    	<h3>2020</h3>
            <font color="#8B0000"><b>Systems biology:</b></font><br>
    
            63.
            <b>Kim, H.</b>, Osteil, P., Humphrey, S., Cinghu, S., Oldfield, A., Patrick, E., Wilkie, E., Peng, G., Suo, S., Jothi, R., Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            Transcriptional network dynamics during the progression of pluripotency revealed by integrative statistical learning.
            <b><i>Nucleic Acids Research</i></b>, 48(4), 1828-1842.
            [<a href="https://doi.org/10.1093/nar/gkz1179">Full Text</a>]
            <br><br>

            62.
            Azimi, A., <b><u>Yang, P.</u></b>, Ali, M., Howard, V., Mann, G., Kaufman, K. & Fernandez-Penas, P. (2020)
            Data independent acquisition proteomic analysis can discriminate between actinic keratosis, Bowen’s disease and cutaneous squamous cell carcinoma.
            <b><i>Journal of Investigative Dermatology</i></b>, 140(1), 212–222.
            [<a href="https://doi.org/10.1016/j.jid.2019.06.128">Full Text</a>]
            <br><br>
            <p class="tab">&bull; <a href="https://doi.org/10.3410/f.736074090.793562184"><img src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif" alt="Access the recommendation on F1000Prime" id="bg" /></a></p>
            <br>

		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
            
            61.
            Cao, Y., <b>Geddes, T.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            Ensemble deep learning in bioinformatics.
            <b><i>Nature Machine Intelligence</i></b>, 2, 500-508.
            [<a href="https://doi.org/10.1038/s42256-020-0217-y">Full Text</a>]
            [<a href="https://rdcu.be/b6jIn">Nature Content Sharing link</a>]
            <br><br>
    
            60.
            Lin, Y., Cao, Y., <b>Kim, H.</b>, Salim, A., Speed, T., Lin, D., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2020)
            scClassify: sample size estimation and multiscale classification of cells using single and multiple reference.
            <b><i>Molecular Systems Biology</i></b>, 16, e9389.
            [<a href="https://doi.org/10.15252/msb.20199389">Full Text</a>]
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/scClassify.html">BioC R package</a>]
            <br><br>
    
            59.
            <b>Kim, H.</b><sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            CiteFuse enables multi-modal analysis of CITE-seq data.
            <b><i>Bioinformatics</i></b>, 36(14), 4137-4143.
            [<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
            [<a href="https://www.biorxiv.org/content/10.1101/854299v1">Biorxiv version</a>]
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>]
            <br><br>

            58.
            Kaur, S., Peters, T., <b><u>Yang, P.</b></u>, Luu, L., Vuong, J., Krycer, J. & O'Donoghue, S. (2020)
            Temporal ordering of omics and multiomic events inferred from time-series data.
            <b><i>npj Systems Biology and Applications</i></b>, 16, 22.
            [<a href="https://www.nature.com/articles/s41540-020-0141-0">Full Text</a>]
            [<a href="https://bit.ly/MinardoModel">Repo</a>]
            <br><br>
    
			57.
			Tang, T., Wu, H., Bao, W., <b><u>Yang, P.</b></u>, Yuan, D., Zhou, B. (2020)
			New parallel algorithms for all pairwise computation on large HPC clusters.
			In <i>the 20th International Conference on Parallel and Distributed Computing, Applications and Technologies <b>(PDCAT)</b></i>, IEEE, 196-201.
			[<a href="https://doi.org/10.1109/PDCAT46702.2019.00045">Full Text</a>]
			<br><br>
			<p class="tab">&bull; Best student paper award</p>
			<br>

		<h3>2019</h3>
			<font color="#8B0000"><b>Systems biology:</b></font><br>
			56.
			Su, Z.<sup>&#10018;</sup>, Burchfield, J.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Humphrey, S., Yang, G., Francis, D., Yasmin, S., Shin, S., Norris, D., 
			Kearney, A., Astore, M., Scavuzzo, J., Fisher-Wellman, K., Wang, Q., Parker, B., Neely, G., Vafaee, F., Chiu, J., Yeo, R., Hogg, P., Fazakerley, D., Nguyen, L., Kuyucak, S. & James, D. (2019)
			Global redox proteome and phosphoproteome analysis reveals redox switch in Akt.
			<b><i>Nature Communications</i></b>, 10, 5486.
			[<a href="https://www.nature.com/articles/s41467-019-13114-4">Full Text</a>]
			<br><br>

			55.
			<b><u>Yang, P.</b></u><sup>&#10018;</sup><sup>&dagger;</sup>, Humphrey, S.<sup>&#10018;</sup><sup>&dagger;</sup>, Cinghu, S.<sup>&#10018;</sup>, Pathania, R., Oldfield, A., Kumar, D., <b>Perera, D.</b>, Yang, J., James, D., Mann, M. & Jothi, R.<sup>&dagger;</sup> (2019)
			Multi-omic profiling reveals dynamics of the phased progression of pluripotency.
			<b><i>Cell Systems</i></b>, 8(5), 427-445.
			[<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30115-2#%20">Full Text</a>]
			[<a href="https://www.biorxiv.org/content/early/2018/09/19/415430.full.pdf">Biorxiv version</a>]
			[<a href="http://stemcellatlas.org/">The Stem Cell Atlas</a>]
			<br><br>
            <p class="tab">&bull; <a href="https://f1000.com/prime/735727099?bd=1" target="_blank"><img src="https://s3.amazonaws.com/cdn.f1000.com/images/badges/badgef1000.gif" alt="Access the recommendation on F1000Prime" id="bg" /></a></p>
			<br>

			54.
			Oldfield, A., Henriques, T., Kumar, D., Burkholder, A., Cinghu, S., Paulet, D., Bennett, B., <b>Yang, P.</b>, Scruggs, B., Lavender, C., Rivals, E., Adelman, K. & Jothi, R. (2019)
			NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nulceosome-depleted region. 
			<b><i>Nature Communications</i></b>, 10, 3072.
			[<a href="https://www.nature.com/articles/s41467-019-10905-7">Full Text</a>]
			<br><br>
			
			53.
			Parker, B.<sup>&#10018;</sup>, Calkin, A.<sup>&#10018;</sup>, Seldin, M.<sup>&#10018;</sup>, Keating, M. Tarling, E., <b><u>Yang, P.</u></b>, Moody, S., Liu, Y., Zerenturk, E.,
			Needham, E., Miller, M., Clifford, B., Morand, P., Watt, M., Meex, R., Peng, K., Lee, R., Jayawardana, K., Pan, C.,
			Mellett, N., Weir, J., Lazarus, R., Lusis, A., Meikle, P., James, D., Vallim, T. & Drew, B. (2019)
			An integrative systems genetic analysis of mammalian lipid metabolism.
			<b><i>Nature</i></b>, 567, 187-193.
			[<a href="https://www.nature.com/articles/s41586-019-0984-y">Full Text</a>]
    		<br><br>
    		
    		52.
    		O'Sullivan, J., Neylon, A., Fahy, E., <b><u>Yang, P.</u></b>, McGorrian, C. & Blake, G. (2019)
    		MiR-93-5p is a novel predictor of coronary in-stent restenosis.
    		<b><i>Heart Asia</i></b>, 11(1), e011134.
    		[<a href="http://dx.doi.org/10.1136/heartasia-2018-011134">Full Text</a>]
			<br><br>
			
    		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
    		
			51. 
			<b>Geddes, T.</b>, <b>Kim, T.</b>, Nan, L., Burchfield, J., Yang, J., Tao, D. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis.
			<b><i>BMC Bioinformatics</i></b>, 20, 660.
			[<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5">Full Text</a>]
			[<a href="https://github.com/gedcom/scCCESS">Repo</a>]
			<br><br>

			50.
			Cao, Y., Lin, Y., Ormerod, J., <b><u>Yang, P.</b></u>, Yang, J. & Lo, K. (2019)
			scDC: single cell differential composition analysis
			<b><i>BMC Bioinformatics</i></b>, 20, 721.
			[<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3211-9">Full Text</a>]
			<a href="https://github.com/SydneyBioX/scdney">Repo</a>]
			<br><br>

			49. 
			<b>Kim, T.</b>, Lo, K., <b>Geddes, T.</b>, <b>Kim, H.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			scReClassify: post hoc cell type classification of single-cell RNA-seq data.
			<b><i>BMC Genomics</i></b>, 20, 913.
			[<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6305-x">Full Text</a>]
			[<a href="https://github.com/SydneyBioX/scReClassify">Repo</a>]
			<br><br>

    		48. Lin, Y., Ghazanfar, S., Strbenac, D., Wang, A., Patrick, E., Lin, D., Speed, T., Yang, J.<sup>&dagger;</sup> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
    		Evaluating stably expressed genes in single cells.
    		<b><i>GigaScience</i></b>, 8(9), giz106.
            [<a href="https://doi.org/10.1093/gigascience/giz106">Full Text</a>]
    		[<a href="http://shiny.maths.usyd.edu.au/SEGs">Interactive tool for SEG download</a>]
    		[<a href="http://gigadb.org/dataset/100637">Giga DB</a>]
    		[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/scHK/scHK_human.xlsx">human HK genes</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/scHK/scHK_mouse.xlsx">mouse HK genes</a>]
    		<br><br>
    		
			47. 
			Lin, Y., Ghazanfar, S., Wang, K., Gagnon-Bartsch, J., Lo, K., Su, X., Han, Z., Ormerod, J., Speed, T., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2019)
			scMerge leverages factor analysis, stable expression, and pseudoreplication to merge multiple single-cell RNA-seq datasets.
			<b><i>Proceedings of the National Academy of Sciences of the United States of America</i></b>, 116(20), 9775-9784.
			[<a href="https://www.pnas.org/content/early/2019/04/25/1820006116">Full Text</a>]
			[<a href="https://bioconductor.org/packages/release/bioc/html/scMerge.html">BioC R package</a>]
			<br><br>		

			46.
			<b>Kim, T.</b>, <b>Chen, I.</b>, Parker, B., Humphrey, S., Crossett, B., Cordwell, S., <b><u>Yang, P.</u></b><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2019)
			QCMAP: An interactive web-tool for performance diagnosis and prediction of LC-MS Systems.
			<b><i>Proteomics</i></b>, 19(3), 1900068.
			[<a href="https://doi.org/10.1002/pmic.201900068">Full Text</a>]
            [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/2019/QCMAP.pdf">PDF</a>]
			[<a href="http://shiny.maths.usyd.edu.au/QCMAP/">QCMAP tool</a>]
			<br><br>
			
			45.
			<b><u>Yang, P.</u></b><sup>&dagger;</sup>, Ormerod, J., Liu, W., <b>Ma, C.</b>, Zomaya, A. & Yang, J. (2019) 
			AdaSampling for positive-unlabeled and label noise learning with bioinformatics applications.
			<b><i>IEEE Transactions on Cybernetics</b></i>, 49(5), 1932-1943.
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/AdaSampling_v11.pdf">PDF</a>],
			[<a href="https://github.com/PengyiYang/AdaSampling">Repo</a>]
			<br><br>

    		44.
			<b>Kim, T.</b>, <b>Chen, I.</b>, Lin, Y., Wang, A., Yang, J. & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2019)
			Impact of similarity metrics on single-cell RNA-seq data clustering.
			<b><i>Briefings in Bioinformatics</i></b>, 20(6), 2316-2326.
			[<a href="https://doi.org/10.1093/bib/bby076">PDF</a>]
			[<a href="https://www.researchgate.net/publication/326769441_Impact_of_similarity_metrics_on_single-cell_RNA-seq_data_clustering">draft</a>]
			[<a href="https://github.com/SydneyBioX/scdney">Repo</a>]
    		<br><br>

			43.
			Ridder, M., Klein, K., Yang, J., <b><u>Yang, P.</u></b>, Lagopoulos, J., Hickie, I., Bennett, M. & Kim, J. (2019)
			An uncertainty visual analytics framework for fMRI functional connectivity.
			<b><i>Neuroinformatics</b></i>, 17(2), 211-223.
			[<a href="https://doi.org/10.1007/s12021-018-9395-8">PDF</a>]
			<br><br>

		<h3>2018</h3>
			<font color="#8B0000"><b>Systems biology:</b></font><br>
			42.
			Fazakerley, DJ., Chaudhuri, R., <b><u>Yang, P.</u></b>, Maghzal, GJ., Thomas, KC., Krycer, JR., Humphrey, SJ., Parker, BL., Fisher-Wellman, KH., 
			Meoli, CC., Hoffman, NJ., Diskin, C., Burchfield, JG., Cowley, MJ., Kaplan, W., Modrusan, Z., Kolumam, G., Yang, JYH, 
			Chen, DL., Samocha-Bonet, D., Greenfield, JR., Hoehn, KL., Stocker, R. & James, DE. (2018) 
			Mitochondrial CoQ deficiency is a common driver of mitochondrial oxidants and insulin resistance.	
			<b><i>eLife</i></b>, 7, e32111.
			[<a href="https://elifesciences.org/articles/32111">Full Text</a>]
    		<br><br>
    		
    	<h3>2017</h3>
    		<font color="#8B0000"><b>Systems biology:</b></font><br>    		
    		41.
    		Cinghu, S.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Kosak, J., Conway, A., Kumar, D., Oldfield, A., Adelman, K. & Jothi, R. (2017)
    		Intragenic enhancers attenuate host gene expression.
			<b><i>Molecular Cell</i></b>, 68(1), 104–117.
			[<a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30658-5">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Attenuator.pdf">PDF</a>]
    		<br><br>
            <p class="tab">&bull; highlighted in <b><i>Nature Reviews Genetics</i></b>, doi:10.1038/nrg.2017.90, 2017. [<a href="http://dx.doi.org/10.1038/nrg.2017.90">fulltext</a>]</p>
			<p class="tab">&bull; also highlighted in <b><i>Nature Reviews Molecular Cell Biology</i></b>, doi:10.1038/nrm.2017.111, 2017. [<a href="http://dx.doi.org/10.1038/nrm.2017.111">fulltext</a>]</p>
    		<br>
    
    		40.
    		Norris, DM.<sup>&#10018;</sup>, <b><u>Yang, P.</u></b><sup>&#10018;</sup>, Krycer JR., Fazakerley DJ, James, DE. & Burchfield JG.<sup>&dagger;</sup> (2017)
    		An improved Akt reporter reveals intra- and inter- cellular heterogeneity and oscillations in signal transduction.
			<b><i>Journal of Cell Science</i></b>, 130, 2757-2766.
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/2757.full.pdf">PDF</a>]
    		<br><br>
    		
    		39.
    		<b><u>Yang, P.</u></b><sup>&dagger;</sup>, Oldfield, A., <b><u>Kim, T.</u></b>, Yang, A., Yang, J. & Ho, J.<sup>&dagger;</sup> (2017)
    		Integrative analysis identifies co-dependent gene expression regulation of BRG1 and CHD7 at distal regulatory sites in embryonic stem cells.
			<b><i>Bioinformatics</i></b>, 33(13), 1916–1920.
			[<a href="https://academic.oup.com/bioinformatics/article/2995820/Integrative-analysis-identifies-co-dependent-gene">Full Text</a>], 
			[<a href="http://pad2.maths.usyd.edu.au/PADv1/">Interactive PAD</a>]
    		<br><br>
    		
    		<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			38.
			<b><u>Yang, P.</u></b>, Liu, W. & Yang, J. (2017) 
			Positive unlabeled learning via wrapper-based adaptive sampling.
			In <i>Proceedings of the 26th International Joint Conference on Artificial Intelligence <b>(IJCAI)</b></i>, 
			Pre-print [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/IJCAI17_Yang.pdf">PDF</a>]
			<br><br>

    
		<h3>2016</h3>

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			37.
			Zheng, X., <b><u>Yang, P.</u></b>, Lackford, B., Bennett, B., Wang, L., Li, H., Wang, Y., Miao, Y., Foley, J., Fargo, D., 
			Jin, Y., Williams, C., Jothi, R. & Hu, G. (2016)
			CNOT3-dependent mRNA deadenylation safeguards the pluripotent state.
			<b><i>Stem Cell Reports</i></b>, 7(5), 897-910.
			[<a href="http://www.cell.com/stem-cell-reports/abstract/S2213-6711(16)30213-2">Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Zheng and Yang Stem Cell Reports 2016.pdf">PDF</a>]
			<br><br>
			
			36.
			Minard, A., Tan, S., <b><u>Yang, P.</b></u>, Fazakerley, D., Domanova, W., Parker, B., Humphrey, S., Jothi, R., Stöckli. J. & James. D. (2016)
			mTORC1 is a major regulatory node in the FGF21 signaling network in adipocytes.
			<b><i>Cell Reports</i></b>, 17(1), 29-36.
			[<a href="https://www.ncbi.nlm.nih.gov/pubmed/27681418">Pubmed</a>]
			<br><br>
			
			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			35.
			<b><u>Yang, P.</b></u>, Patrick, E., Humphrey, S., Ghazanfar, S., James, D., Jothi, R. & Yang, J. (2016). 
			KinasePA: Phosphoproteomics data annotation using hypothesis driven kinase perturbation analysis.
			<b><i>Proteomics</i></b>, 16(13), 1868-1871.
			[<a href="https://doi.org/10.1002/pmic.201600068">Full Text</a>], 
            [<a href="https://github.com/PYangLab/directPA">Github Repo & tutorial</a>], 
            [<a href="https://cran.r-project.org/package=directPA">CRAN R package</a>]
			<br><br>
			
			34.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Humphrey, S., James, D., Yang, J. & Jothi, R.<sup>&dagger;</sup> (2016). 
			Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data.
			<b><i>Bioinformatics</i></b>, 32(2), 252-259.
       		[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26395771">Pubmed</a>], 
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Bioinformatics2016positiveUnlabelLearning.pdf">PDF</a>],
       		[<a href="https://github.com/PYangLab/KSP-PUEL">Github Repo</a>]
       		<br><br>
            
			33.
			Lu, C., Wang, J., Zhang, Z., <b><u>Yang, P.</b></u> & Yu, G. (2016).
			NoisyGOA: Noisy GO annotations prediction using taxonomic and semantic similarity.
			<b><i>Computational Biology and Chemistry</i></b>, 65, 203-211.
			<br><br>
					
			32.
			Domanova, W., Krycer, J., Chaudhuri, R., <b><u>Yang, P.</b></u>, Vafaee, F., Fazakerley, D., Humphrey, S., James, D. 
			& Kuncic, Z., (2016). Unraveling kinase activation dynamics using kinase-substrate relationships from temporal large-scale phosphoproteomics studies.
			<b><i>PLoS One</i></b>, 11(6), e0157763.
            <br><br>
    
			<!--
			<font color="#FF0000">Practical contribution:</font><font color="#000000"> 
			this study provides a graphical user interface (GUI)-based syste for predicting substrates of kinases by integrating dynamic phosphoproteomics data with kinase recognition motif. <br>		
		
			<font color="#FF0000">Code available from:</font><font color="#000000">
	    	</font><a href="https://github.com/PengyiYang/KSP-PUEL">https://github.com/PengyiYang/KSP-PUEL</a><font color="#000000"><br>
			-->

		<h3>2015</h3>	

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			31.
			Pathania, R., Ramachandran, S., Elangovan, S., Padia, R., <b><u>Yang, P.</b></u>, Cinghu, S., Veeranan-Karmegam, R., Fulzele, S., 
			Pei, L., Chang, C., Choi, J., Shi, H., Manicassamy, S., Prasad, P., Sharma, S., Ganapathy, V., Jothi, R. & Thangaraju, M. (2015). 
			DNMT1 is essential for mammary and cancer stem cell maintenance and tumorigenesis. 
			<b><i>Nature Communications</i></b>, 6, 6910.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25908435">Pubmed</a>]
			<br><br>
			
			30.
			Hoffman, N., Parker, B., Chaudhuri, R., Fisher-Wellman, K., Kleinert, M., Humphrey, S., <b><u>Yang, P.</b></u>, Holliday, M., Trefely, S., 
			Fazakerley, D., Stockli, J., Burchfield, J., Jensen, T., Jothi, R., Kiens, B., Wojtaszewski, J., Richter, E. & James, D. (2015). 
			Global phosphoproteomic analysis of human skeletal muscle reveals a network of exercise-regulated kinases and AMPK substrates. 
			<b><i>Cell Metabolism</i></b>, 22(5), 922-935.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/26437602">Pubmed</a>]
			<br><br>
			
			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			29.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zheng, X., Jayaswal, V., Hu, G., Yang, J. & Jothi, R. (2015). 
			Knowledge-based analysis for detecting key signaling events from time-series phosphoproteomics data. 
			<b><i>PLoS Computational Biology</i></b>, 11(8), e1004403.
			[<a href="https://doi.org/10.1371/journal.pcbi.1004403">Full Text</a>],
            [<a href="https://github.com/PYangLab/ClueR">Github Repo</a>],
            [<a href="https://CRAN.R-project.org/package=ClueR">CRAN R package</a>]
            <br><br>
    
		<h3>2014</h3>

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			28.
			Oldfield, A.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Conway, A., Cinghu, S., Freudenberg, J., Yellaboina, S. & Jothi, R. (2014). 
			Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors. 
			<b><i>Molecular Cell</i></b>, 55(5), 708-722.
			[<a href="https://doi.org/10.1016/j.molcel.2014.07.005">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/NFY.pdf">PDF</a>]
			<br><br>
			
			27.
			Ma, X., <b><u>Yang, P.</b></u>, Kaplan, W., Lee, B., Wu, L., Yang, J., Yasunaga, M., Sato, K., Chisholm, D. & James, D. (2014). 
			ISL1 regulates peroxisome proliferator-activated receptor &gamma; activation and early adipogenesis via bone morphogenetic protein 
			4-dependent and -independent mechanisms.
			<b><i>Molecular and Cellular Biology</i></b>, 34(19), 3607-3617.
			[<a href="https://doi.org/10.1128/MCB.00583-14">Full text</a>]
			<br><br>
			
			26.
			Lackford, B., Yao, C., Charles, G., Weng, L., Zheng, X., Choi, E., Xie, X., Wan, J., Xing, Y., Freudenberg, J., 
			<b><u>Yang, P.</b></u>, Jothi, R., Hu, G. & Shi, Y. (2014). 
			Fip1 regulates mRNA alternative polyadenylation to promote stem cell self‐renewal. 
			<b><i>EMBO Journal</i></b>, 33(8), 878-889.
			[<a href="https://doi.org/10.1002/embj.201386537">Full Text</a>]
			<br><br>
			
			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			25.
			<b><u>Yang, P.</b></u><sup>&#10018;</sup>, Patrick, E.<sup>&#10018;</sup>, Tan, S., Fazakerley, D., Burchfield, J., Gribben, C., Prior, M., James, D. & Yang, J. (2014). 
			Direction pathway analysis of large-scale proteomics data reveals novel features of the insulin action pathway. 
			<b><i>Bioinformatics</i></b>, 30(6), 808-814.
			[<a href="https://doi.org/10.1093/bioinformatics/btt616">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Direction.pdf">PDF</a>], 
            [<a href="https://github.com/PYangLab/directPA">Github Repo & tutorial</a>], 
            [<a href="https://cran.r-project.org/package=directPA">CRAN R package</a>]
			<br><br>
			
			24.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Yoo, P., Fernando, J., Zhou, B., Zhang, Z. & Zomaya, A. (2014). 
			Sample subset optimization techniques for imbalanced and ensemble learning problems in bioinformatics applications. 
			<b><i>IEEE Transactions on Cybernetics</i></b>, 44(3), 445-455.
			[<a href="https://doi.org/10.1109/TCYB.2013.2257480">Full Text</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/SSO for imbalanced and ensemble learning.pdf">PDF</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/software/Sampling.html">Software</a>]
            <br><br>
    
		<h3>2013</h3>

			<font color="#8B0000"><b>Systems biology:</b></font><br>
			23.
			Humphrey, S., Yang, G., <b><u>Yang, P.</b></u>, Fazakerley, D. J., Stöckli, J., Yang, J. & James, D. (2013). 
			Dynamic adipocyte phosphoproteome reveals that Akt directly regulates mTORC2. 
			<b><i>Cell Metabolism</i></b>, 17(6), 1009-1020.
			[<a href="https://doi.org/10.1016/j.cmet.2013.04.010">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Adipocyte phosphoproteome.pdf">PDF</a>]
			<br><br>

			<font color="#8B0000"><b>Methodology and tools:</b></font><br>
			22.
			<b><u>Yang, P.</b></u>, Liu, W., Zhou, B., Chawla, S. & Zomaya, A. (2013). 
			Ensemble-based wrapper methods for feature selection and class imbalance learning. 
			In <i>Proceedings of the 17th Pacific-Asia Conference on Knowledge Discovery and Data Mining <b>(PAKDD)</b></i>, 
			Lecture Notes in Artificial Intelligence 7818, Springer Berlin Heidelberg, 544-555.
			[<a href="https://doi.org/10.1007/978-3-642-37453-1_45">Full Text</a>]
			<br><br>
			
			21.
			<b><u>Yang, P.</b></u>, Yang, J., Zhou, B. & Zomaya. A. (2013).
			Stability of feature selection algorithms and ensemble feature selection methods in bioinformatics. 
			In <i>Biological Knowledge Discovery Handbook: Preprocessing, Mining and Postprocessing of Biological Data</i>, 
			Wiley, New Jersey, USA, 333-352.
            [<a href="https://doi.org/10.1002/9781118617151.ch14">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Yang_et_al_stability.pdf">PDF</a>]
            <br><br>
			
		<h3>2012</h3>

			20.
			<b><u>Yang, P.</b></u>, Humphrey, S., Fazakerley, D., Prior, M., Yang, G., James, D. & Yang, J. (2012). 
			Re-fraction: a machine learning approach for deterministic identification of protein homologues and splice variants in 
			large-scale MS-based proteomics. 
			<b><i>Journal of Proteome Research</i></b>, 11(5), 3035-3045.
			[<a href="https://doi.org/10.1021/pr300072j">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/pr300072j.pdf">PDF</a>]
			<br><br>
			
			19.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Ma, J., Wang, P., Zhu, Y., Zhou, B. & Yang, J. (2012). 
			Improving X! Tandem on peptide identification from mass spectrometry by self-boosted Percolator. 
			<b><i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i></b>, 9(5), 1273-1280.
			[<a href="https://doi.org/10.1109/TCBB.2012.86">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/self-boosted percolator for xtandem.pdf">PDF</a>]
			<br><br>	
			
			18.
			Wang, P., <b><u>Yang, P.</b></u> & Yang, J. (2012). 
			OCAP: an open comprehensive analysis pipeline for iTRAQ. 
			<b><i>Bioinformatics</i></b>, 28(10), 1404-1405.
			[<a href="https://doi.org/10.1093/bioinformatics/bts150">Full Text</a>]
            <br><br>
			
		<h3>2011</h3>

			17.
			<b><u>Yang, P.</b></u><sup>&#10018;, &dagger;</sup>, Ho, J.<sup>&#10018;</sup>, Yang, J. & Zhou, B. (2011). 
			Gene-gene interaction filtering with ensemble of filters. 
			<b><i>BMC Bioinformatics</i></b>, 12, S10.
			[<a href="https://doi.org/10.1186/1471-2105-12-S1-S10">Full Text</a>]
			<br><br>
			
			16.
			<b><u>Yang, P.</b></u>, Zhang, Z., Zhou, B. & Zomaya, A. (2011). 
			Sample subset optimization for classifying imbalanced biological data. 
			In <i>Proceedings of the 15th Pacific-Asia Conference on Knowledge Discovery and Data Mining <b>(PAKDD)</b></i>, 
			Lecture Notes in Artificial Intelligence 6635, Springer Berlin Heidelberg, 333-344.
			[<a href="https://doi.org/10.1007/978-3-642-20847-8_28">Text</a>]
            <br><br>

		<h3>2010</h3>

			15.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Ho, J., Zomaya, A. & Zhou, B. (2010). 
			A genetic ensemble approach for gene-gene interaction identification. 
			<b><i>BMC Bioinformatics</i></b>, 11(1), 524.
			[<a href="https://doi.org/10.1186/1471-2105-11-524">Full Text</a>]
			<br><br>
			
			14.
			Wang, P., <b><u>Yang, P.</b></u>, Arthur, J. & Yang, J. (2010). 
			A dynamic wavelet-based algorithm for pre-processing tandem mass spectrometry data. 
			<b><i>Bioinformatics</i></b>, 26(18), 2242-2249.
			[<a href="https://doi.org/10.1093/bioinformatics/btq403">Full Text</a>]
			<br><br>
			
			13.
			Yoo, P., Ho, Y., Ng, J., Charleston, M., Saksena, N., <b><u>Yang, P.</b></u> & Zomaya, A. (2010). 
			Hierarchical kernel mixture models for the prediction of AIDS disease progression using HIV structural gp120 profiles. 
			<b><i>BMC Genomics</i></b>, 11, S22.
			[<a href="https://doi.org/10.1186/1471-2164-11-s4-s22">Full Text</a>]
			<br><br>
			
			12.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zhang, Z., Zhou, B. & Zomaya, A. (2010). 
			A clustering based hybrid system for biomarker selection and sample classification of mass spectrometry data. 
			<b><i>Neurocomputing</i></b>, 73(13), 2317-2331.
			[<a href="https://doi.org/10.1016/j.neucom.2010.02.022">Full Text</a>]
			<br><br>
			
			11.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zhou, B., Zhang, Z. & Zomaya, A. (2010). 
			A multi-filter enhanced genetic ensemble system for gene selection and sample classification of microarray data. 
			<b><i>BMC Bioinformatics</i></b>, 11, S5.
			[<a href="https://doi.org/10.1186/1471-2105-11-S1-S5">Full Text</a>]
			<br><br>
			
			10.
			<b><u>Yang, P.</b></u>, Yang, J., Zhou, B. & Zomaya, A. (2010). 
			A review of ensemble methods in bioinformatics. 
			<b><i>Current Bioinformatics</i></b>, 5(4), 296-308.
			[<a href="https://doi.org/10.2174/157489310794072508">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/EnsembleBioinformatics-v6.pdf">PDF</a>]
			<br><br>			
			
			9.
			Li, L., <b><u>Yang, P.</b></u>, Ou, L., Zhang, Z. & Cheng, P. (2010). 
			Genetic algorithm-based multi-objective optimisation for QoS-aware web services composition. 
			In <i>Proceedings of the 4th International Conference on Knowledge Science, Engineering and Management <b>(KSEM)</b></i>, 
			Lecture Notes in Computer Science 6291, Springer Berlin Heidelberg, 549-554.
			[<a href="https://doi.org/10.1007/978-3-642-15280-1_50">Full Text</a>]
            <br><br>
			
		<h3>2009</h3>

			8.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Xu, L., Zhou, B., Zhang, Z. & Zomaya, A. (2009). 
			A particle swarm based hybrid system for imbalanced medical data sampling. 
			<b><i>BMC Genomics</i></b>, 10, S34.
			[<a href="https://doi.org/10.1186/1471-2164-10-S3-S34">Full Text</a>]
			<br><br>
			
			7.
			Zhang, Z., <b><u>Yang, P.</b></u>, Wu, X. & Zhang, C. (2009). 
			An agent-based hybrid system for microarray data analysis. 
			<b><i>IEEE Intelligent Systems</i></b>, 24(5), 53-63.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30029412/zhang-agentbasedhybridsystem-2009.pdf">PDF</a>]
			<br><br>
			
			6.
			<b><u>Yang, P.</b></u><sup>&dagger;</sup> & Zhang, Z. (2009). An embedded two-layer feature selection approach for microarray data analysis. 
			<b><i>IEEE Intelligent Informatics Bulletin</i></b>, 10(1), 24-32.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30029413/zhang-embeddedtwolayer-2009.pdf">PDF</a>]
			<br><br>
			
			5.
			<b><u>Yang, P.</b></u>, Tao, L., Xu, L. & Zhang, Z. (2009). 
			Multiagent framework for bio-data mining. 
			In <i>Proceedings of the 4th Rough Sets and Knowledge Technology <b>(RSKT)</b></i>, 
			Lecture Notes in Computer Science 5589, Springer Berlin Heidelberg, 200-207.
			[<a href="https://doi.org/10.1007/978-3-642-02962-2_25">Text</a>]
			<br><br>

			
		<h3>2008</h3>

			4.
			Zhang, Z. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2008). 
			An ensemble of classifiers with genetic algorithm-based feature selection. 
			<b><i>IEEE Intelligent Informatics Bulletin</i></b>, 9(1), 18-24.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30017964/zhang-anemembleofclassifiers-2008.pdf">PDF</a>]
			<br><br>
			
			3.
			<b><u>Yang, P.</b></u> & Zhang, Z. (2008). 
			A clustering based hybrid system for mass spectrometry data analysis. 
			In <i>Proceedings of the 3rd Pattern Recognition in Bioinformatics <b>(PRIB)</b></i>, 
			Lecture Notes in Bioinformatics 5265, Springer Berlin Heidelberg, 98-109.
			[<a href="https://doi.org/10.1007/978-3-540-88436-1_9">Full Text</a>]
			<br><br>
			<p class="tab">&bull; International student travel award</p>
			<br>
						
			2. 
			<b><u>Yang, P.</b></u> & Zhang, Z. (2008).
			A hybrid approach to selecting susceptible single nucleotide polymorphisms for complex disease analysis. 
			In <i>Proceedings of BioMedical Engineering and Informatics <b>(BMEI)</b></i>, IEEE, 214-218.
			[<a href="http://dro.deakin.edu.au/eserv/DU:30018221/zhang-ahybridapproach-2008.pdf">PDF</a>]
            <br><br>
			
		<h3>2007</h3>
			1. <b><u>Yang, P.</b></u> & Zhang, Z. (2007). 
			Hybrid methods to select informative gene sets in microarray data classification. 
			In <i>Proceedings of the 20th Australian Joint Conference on Artificial Intelligence <b>(AI)</b></i>, 
			Lecture Notes in Artificial Intelligence 4830, Springer Berlin Heidelberg, 811-815.
			[<a href="https://doi.org/10.1007/978-3-540-76928-6_97">Text</a>]	
            <br><br>
    
        <hr>
		
        <h2 class="publications">Others</h2>
     	Ensemble methods and hybrid algorithms for computational and systems biology (PhD thesis, Pengyi Yang) [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/thesis/Pengyi-phdthesis-filed.pdf">PDF</a>]
        <br><br>
    
        Computational tools for the integrative analysis of multi-omics data to decipher trans-omics networks (PhD thesis, Hani Jieun Kim) [<a href="https://hdl.handle.net/2123/28524">PDF</a>]
        <br><br>
    
        Development of statistical methods for integrative omics analysis in precision medicine (PhD thesis, Taiyun Kim) [<a href="https://hdl.handle.net/2123/28838">PDF</a>]
        <br><br>
</div>
