---
layout: default
---

<div class="container">

<div class="section row">
	<div class="col-md-12">
	<h1 class="heading"><b>Research summary</b></h1><br>
        Cells are the fundamental building blocks of all life on earth. In multicellular organisms, these cellular building blocks come in a plethora of types and serve distinct functions to coordinate and support the homeostasis of an organism. Intriguingly, all cell types of an organism share the same genetic code and originate from the same group of pluripotent <b>stem cells</b> that have undergone different cellular fates during development.
        <br><br>
        Research from the last few decades has established that cell identity and cell-fate decisions are determined by underlying molecular programs such as gene transcription and translation, and the intra- and extra-cellular networks they form. Understanding how such programs/networks regulate and interact with one another in a spatial-temporal manner is critical for gaining insight into the complex nature of multicellular development. To this end, our research takes a holistic approach and seeks to understand the molecular 'trans-regulatory networks', comprised of cell signalling, transcriptional, translational, and (epi)genomic regulations, in controlling cell identity and cell-fate decisions. In particular, we develop computational and statistical models to reconstruct each layer of the trans-regulatory networks (e.g. signalling networks, transcriptional networks) and characterise their cross-talk in stem cells during their differentiation to specialised cell types. By employing a multidisiciplinary approach that combines 'dry' (computation) and 'wet' (laboratory) studies at the systems level, we aim to address the following research questions:<br><br>

        <ul>
            <li>How do different layers of trans-regulatory networks coordinately regulate cell identity and cell fate-decisions?</li>
            <li>How can we accurately predict cell identity and cell fate-decisions based on their trans-regulatory networks?</li>
            <li>How can we modulate trans-regulatory networks to direct cell fate-decisions for applications such as stem cell therapy?</li>
        </ul>
	</div>
</div>        
<hr>
 
<! -- Direction 1 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <h3 class="projects">Pluripotency progression in stem cells and lineage specification</h3>
        <br>
        Pluripotency defines the potential of stem cells to differentiate into somatic and germ cells. The bookends of the pluripotency spectrum, naïve and primed states, are well studied but the progression of pluripotency from naïve to primed states remain to be fully characterised and their relationship with cell lineage specification remains to be investigated. We use multi-layered trans‐omics approach to study pluripotency transition from naïve to formative, an intermediate state, and then primed state. We developing methods that allows the reconstruction of the underlying trans-omic networks and correlating pluripotent transition to in vivo embryos during cell lineage specification.
    </div>
    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/MultiSummary.jpg"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        <b><u>Yang, P.</b></u><sup>&#10018;</sup><sup>&dagger;</sup>, Humphrey, S.<sup>&#10018;</sup><sup>&dagger;</sup>, Cinghu, S.<sup>&#10018;</sup>, Pathania, R., Oldfield, A., Kumar, D., <b>Perera, D.</b>, Yang, J., James, D., Mann, M. & Jothi, R.<sup>&dagger;</sup> (2019)
        Multi-omic profiling reveals dynamics of the phased progression of pluripotency.
        <b><i>Cell Systems</i></b>, 8(5), 427-445.
        [<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30115-2#%20">Full Text</a>]
        [<a href="https://www.biorxiv.org/content/early/2018/09/19/415430.full.pdf">Biorxiv version</a>]
        [<a href="http://stemcellatlas.org/">The Stem Cell Atlas</a>]
        <br><br>    

        <b>Kim, H.</b>, Osteil, P., Humphrey, S., Cinghu, S., Oldfield, A., Patrick, E., Wilkie, E., Peng, G., Suo, S., Jothi, R., Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
        Transcriptional network dynamics during the progression of pluripotency revealed by integrative statistical learning.
        <b><i>Nucleic Acids Research</i></b>, 48(4), 1828-1842.
        [<a href="https://doi.org/10.1093/nar/gkz1179">Full Text</a>]
        <br><br>

        Zheng, X., <b><u>Yang, P.</u></b>, Lackford, B., Bennett, B., Wang, L., Li, H., Wang, Y., Miao, Y., Foley, J., Fargo, D., 
        Jin, Y., Williams, C., Jothi, R. & Hu, G. (2016)
        CNOT3-dependent mRNA deadenylation safeguards the pluripotent state.
        <b><i>Stem Cell Reports</i></b>, 7(5), 897-910.
        [<a href="http://www.cell.com/stem-cell-reports/abstract/S2213-6711(16)30213-2">Text</a>],
        [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Zheng and Yang Stem Cell Reports 2016.pdf">PDF</a>]
        <br><br>
    </div>
</div>
<br>

<! -- Direction 2 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <br>
        <h3 class="projects">Stem cell and organoid engineering for regenerative medicine</h3>
        <br>
        The holy grail in stem cell and organoid research is to generate stem cell-derived cells, tissues, and organs that can structurally and functionally replace any specific cell populations, tissues, and organs that are compromised by disease, injury, or ageing. While tremendous progress has been made in stem cell-based therapy, attempts to direct stem/progenitor cells to clinically useful cell types and tissues have often been fraught with imprecise and incomplete differentiation. We study and characterise stem cells and organoids using bulk and single-cell multi-omics approaches for realising their potential for regenerative medicine. Discovery made from these works will provide both basic and translational knowledge for stem cell and organoid engineering for tissue/organ regeneration.
    </div>
                    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/.jpg"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        <b>Kim, H.</b>, O'Hara-Wright, M., <b>Kim, D.</b>, Loi, T., Lim, B., Jamieson, R., Gonzalez-Cordero, A.<sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2023)
        Comprehensive characterization of fetal and mature retinal cell identity to assess the fidelity of retinal organoids.
        <b><i>Stem Cell Reports</i></b>, 18(1), 175-189.
        [<a href="https://doi.org/10.1016/j.stemcr.2022.12.002">Full Text</a>]
        [<a href="https://shiny.maths.usyd.edu.au/Eikon/">Eikon Shiny Web Server</a>]
        <br><br> 

        Fernando, M., Lee, S., Wark, J., <b>Xiao, D.</b>, Lim, B., O'Hara-Wright, M., <b>Kim, H.</b>, Smith, G., Wong, T., Teber, E., Ali, R., <b><u>Yang, P.</u></b>, Graham, M. & Gonzalez-Cordero, A. (2022)
        Differentiation of brain and retinal organoids from confluent cultures of pluripotent stem cells connected by nerve-like axonal projections of optic origin.
        <b><i>Stem Cell Reports</i></b>, 17(6), 1476-1492.
        [<a href="https://doi.org/doi:10.1016/j.stemcr.2022.04.003">Full Text</a>]
        <br><br>

        <b>Xiao, D.</b>, Caldow, M., <b>Kim, H.</b>, Blazev, R., Koopman, R., Manandi, D., Parker, B. <sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2022)
        Time-resolved phosphoproteome and proteome analysis reveals kinase signalling on master transcription factors during myogenesis.
        <b><i>iScience</i></b>, 25(6), 104489.
        [<a href="https://doi.org/10.1016/j.isci.2022.104489">Full Text</a>]
        <br><br>
        
    </div>    
</div>
<br>

<! -- Direction 3 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <br>
        <h3 class="projects">Machine learning for computational systems biology</h3>
        <br>

    </div>
                    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/DALLE 2023.png"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        Cao, Y., <b>Geddes, T.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
        Ensemble deep learning in bioinformatics.
        <b><i>Nature Machine Intelligence</i></b>, 2, 500-508.
        [<a href="https://doi.org/10.1038/s42256-020-0217-y">Full Text</a>]
        [<a href="https://rdcu.be/b6jIn">Nature Content Sharing link</a>]
        <br><br>

        <b>Geddes, T.</b>, <b>Kim, T.</b>, Nan, L., Burchfield, J., Yang, J., Tao, D. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
        Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis.
        <b><i>BMC Bioinformatics</i></b>, 20, 660.
        [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5">Full Text</a>]
        [<a href="https://github.com/gedcom/scCCESS">Repo</a>]
        <br><br>

        Lin, Y., Cao, Y., <b>Kim, H.</b>, Salim, A., Speed, T., Lin, D., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2020)
        scClassify: sample size estimation and multiscale classification of cells using single and multiple reference.
        <b><i>Molecular Systems Biology</i></b>, 16, e9389.
        [<a href="https://doi.org/10.15252/msb.20199389">Full Text</a>]
        [<a href="http://www.bioconductor.org/packages/release/bioc/html/scClassify.html">BioC R package</a>]
        <br><br>
        
    </div>    
    
</div>
<br>

<! -- Direction 4 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <br>
        <h3 class="projects">Data analytics for high-throughput single-cell multi-omics</h3>
        <br>

    </div>
                    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/.jpg"/ Align="Left">
    </div>
    <br>
    
</div>
<br>

</div>
















