---
layout: default
---

<div class="container">
<br>
 
<! -- Direction 1 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <h3 class="projects">Pluripotency progression in stem cells and lineage specification</h3>
        <br>
        Pluripotency defines the potential of stem cells to differentiate into somatic and germ cells. The bookends of the pluripotency spectrum, naïve and primed states, are well studied but the progression of pluripotency from naïve to primed states remain to be fully characterised and their relationship with cell lineage specification remains to be investigated. We use multi-layered trans‐omics approach to study pluripotency transition from naïve to formative, an intermediate state, and then primed state. We developing methods that allows the reconstruction of the underlying trans-omic networks and correlating pluripotent transition to in vivo embryos during cell lineage specification.
    </div>
    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/MultiSummary.jpg"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        <b><u>Yang, P.</b></u><sup>&#10018;</sup><sup>&dagger;</sup>, Humphrey, S.<sup>&#10018;</sup><sup>&dagger;</sup>, Cinghu, S.<sup>&#10018;</sup>, Pathania, R., Oldfield, A., Kumar, D., <b>Perera, D.</b>, Yang, J., James, D., Mann, M. & Jothi, R.<sup>&dagger;</sup> (2019)
        Multi-omic profiling reveals dynamics of the phased progression of pluripotency.
        <b><i>Cell Systems</i></b>, 8(5), 427-445.
        [<a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(19)30115-2#%20">Full Text</a>]
        [<a href="https://www.biorxiv.org/content/early/2018/09/19/415430.full.pdf">Biorxiv version</a>]
        [<a href="http://stemcellatlas.org/">The Stem Cell Atlas</a>]
        <br><br>    

        <b>Kim, H.</b>, Osteil, P., Humphrey, S., Cinghu, S., Oldfield, A., Patrick, E., Wilkie, E., Peng, G., Suo, S., Jothi, R., Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
        Transcriptional network dynamics during the progression of pluripotency revealed by integrative statistical learning.
        <b><i>Nucleic Acids Research</i></b>, 48(4), 1828-1842.
        [<a href="https://doi.org/10.1093/nar/gkz1179">Full Text</a>]
        <br><br>

        Zheng, X., <b><u>Yang, P.</u></b>, Lackford, B., Bennett, B., Wang, L., Li, H., Wang, Y., Miao, Y., Foley, J., Fargo, D., 
        Jin, Y., Williams, C., Jothi, R. & Hu, G. (2016)
        CNOT3-dependent mRNA deadenylation safeguards the pluripotent state.
        <b><i>Stem Cell Reports</i></b>, 7(5), 897-910.
        [<a href="http://www.cell.com/stem-cell-reports/abstract/S2213-6711(16)30213-2">Text</a>],
        [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Zheng and Yang Stem Cell Reports 2016.pdf">PDF</a>]
        <br><br>
    </div>
</div>
<br>

<! -- Direction 2 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <h3 class="projects">Stem cell and organoid engineering for regenerative medicine</h3>
        <br>
        A key goal of stem cell and organoid research is to generate stem cell-derived cells, tissues, and organs that can structurally and functionally replace any specific cell populations, tissues, and organs that are compromised by disease, injury, or ageing. While tremendous progress has been made in stem cell-based therapy, attempts to direct stem/progenitor cells to clinically useful cell types and tissues have often been fraught with imprecise and incomplete differentiation. We characterise stem cells and organoids using bulk and single-cell multi-omics approaches with the aim of realising their potential for regenerative medicine. Discovery made from these works will provide both basic and translational knowledge for stem cell and organoid engineering for tissue/organ regeneration.
    </div>

    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/retinalOrganoids.jpg"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        <b>Kim, H.</b>, O'Hara-Wright, M., <b>Kim, D.</b>, Loi, T., Lim, B., Jamieson, R., Gonzalez-Cordero, A.<sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2023)
        Comprehensive characterization of fetal and mature retinal cell identity to assess the fidelity of retinal organoids.
        <b><i>Stem Cell Reports</i></b>, 18(1), 175-189.
        [<a href="https://doi.org/10.1016/j.stemcr.2022.12.002">Full Text</a>]
        [<a href="https://shiny.maths.usyd.edu.au/Eikon/">Eikon Shiny Web Server</a>]
        <br><br> 

        Fernando, M., Lee, S., Wark, J., <b>Xiao, D.</b>, Lim, B., O'Hara-Wright, M., <b>Kim, H.</b>, Smith, G., Wong, T., Teber, E., Ali, R., <b><u>Yang, P.</u></b>, Graham, M. & Gonzalez-Cordero, A. (2022)
        Differentiation of brain and retinal organoids from confluent cultures of pluripotent stem cells connected by nerve-like axonal projections of optic origin.
        <b><i>Stem Cell Reports</i></b>, 17(6), 1476-1492.
        [<a href="https://doi.org/doi:10.1016/j.stemcr.2022.04.003">Full Text</a>]
        <br><br>

        <b>Xiao, D.</b>, Caldow, M., <b>Kim, H.</b>, Blazev, R., Koopman, R., Manandi, D., Parker, B. <sup>&dagger;</sup> & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2022)
        Time-resolved phosphoproteome and proteome analysis reveals kinase signalling on master transcription factors during myogenesis.
        <b><i>iScience</i></b>, 25(6), 104489.
        [<a href="https://doi.org/10.1016/j.isci.2022.104489">Full Text</a>]
        <br><br>
        
    </div>    
</div>
<br>

<! -- Direction 3 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <h3 class="projects">Machine learning for computational systems biology</h3>
        <br>
        Machine learning has become a key driver behind the revolution in computational systems biology. Leveraging on large-scale multi-omics data generated from high-througput biotechnological platforms, we develop specialised machine learning models, such as ensemble learning, deep learning, and multi-task learning models, to extract biological knowledge embedded in these data that are otherwise inaccessible. We aim to make machine learning more robust, flexiable, and interpretable for computational systems biology.
    </div>
                    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/EnsembleDeepLearning.png"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        Cao, Y., <b>Geddes, T.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
        Ensemble deep learning in bioinformatics.
        <b><i>Nature Machine Intelligence</i></b>, 2, 500-508.
        [<a href="https://doi.org/10.1038/s42256-020-0217-y">Full Text</a>]
        [<a href="https://rdcu.be/b6jIn">Nature Content Sharing link</a>]
        <br><br>

        <b>Geddes, T.</b>, <b>Kim, T.</b>, Nan, L., Burchfield, J., Yang, J., Tao, D. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
        Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis.
        <b><i>BMC Bioinformatics</i></b>, 20, 660.
        [<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5">Full Text</a>]
        [<a href="https://github.com/gedcom/scCCESS">Repo</a>]
        <br><br>

        Lin, Y., Cao, Y., <b>Kim, H.</b>, Salim, A., Speed, T., Lin, D., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2020)
        scClassify: sample size estimation and multiscale classification of cells using single and multiple reference.
        <b><i>Molecular Systems Biology</i></b>, 16, e9389.
        [<a href="https://doi.org/10.15252/msb.20199389">Full Text</a>]
        [<a href="http://www.bioconductor.org/packages/release/bioc/html/scClassify.html">BioC R package</a>]
        <br><br>
        
    </div>    
    
</div>
<br>

<! -- Direction 4 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <h3 class="projects">Data analytics for spatial and single-cell multi-omics</h3>
        <br>
        Data analytics for spatial and single-cell multi-omics refers to the analysis and integration of multiple types of genomic data (e.g. transcriptomics, epigenomics, and proteomics) generated from high-throughput single-cell or spatial technologies to gain a comprehensive understanding of cellular and spatiotemporal biological processes. The goal is to identify patterns and relationships between the different omics layers and to link them to cellular functions and phenotypes. We specialise in developing statistical methods and data analytics solutions such as dimensionality reduction, clustering, classification, and visualisation tools, as well as integration algorithms to combine the different omics data types into a unified view for characterising cellular and biological systems.
    </div>
                    
    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/Cepo2021.png"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>

        <b>Kim, H.</b>, Wang, K., <b>Chen, C.</b>, Lin, Y., Tam, PPL., Lin, D., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
        Uncovering cell identity through differential stability with Cepo.
        <b><i>Nature Computational Science</i></b>, 1, 784-790.
        [<a href="https://doi.org/10.1038/s43588-021-00172-2">Full Text</a>]
        [<a href="https://rdcu.be/cDyXi">Nature Content Sharing link</a>]
        [<a href="https://bioconductor.org/packages/release/bioc/html/Cepo.html">BioC R package</a>]
        <br><br>

        <b>Kim, H.</b><sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
        CiteFuse enables multi-modal analysis of CITE-seq data.
        <b><i>Bioinformatics</i></b>, 36(14), 4137-4143.
        [<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
        [<a href="https://www.biorxiv.org/content/10.1101/854299v1">Biorxiv version</a>]
        [<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>]
        <br><br>
        
        <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
        PhosR enables processing and functional analysis of phosphoproteomic data.
        <b><i>Cell Reports</i></b>, 34(8), 108771.
        [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
        [<a href="https://pyanglab.github.io/PhosR/">BioC R package</a>]
        [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
        <br><br>

    </div>
    
</div>
<br>
    
<! -- Direction 5 -->
<div class="section row" style="background-color: #F9F9F9;">
    <div class="col-md-9">
        <h3 class="projects">Transcriptional and epigenomic regulations in stem cells</h3>
        <br>
        Transcriptional regulation involves the activation or repression of gene expression through the binding of transcription factors and cofactors to specific and accessible genomics regions, while epigenomic regulation modulates such genomic accessibility through various changes such as the histone modifications (e.g. acetylation and methylation). In stem cells, transcriptional and epigenomic regulations are crucial for maintaining their pluripotency and differentiation potential. Disruptions in these regulatory mechanisms can therefore lead to stem cell dysfunction and the development of various diseases. We aim to understand the intricate interplay between transcriptional and epigenomic regulation in stem cells and during their differentiation into terminal cell types. Knowledge acquire from these research is crucial for the development of stem cell-based regenerative therapies.
    </div>

    <div class="image col-md-3">
        <img class="img-responsive" src="/images/projects/NFY2014.jpg"/ Align="Left">
    </div>
    <br>
    
    <div class="col-md-12">
        Related publications:<br><br>
        
        Oldfield, A.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Conway, A., Cinghu, S., Freudenberg, J., Yellaboina, S. & Jothi, R. (2014). 
        Histone-fold domain protein NF-Y promotes chromatin accessibility for cell type-specific master transcription factors. 
        <b><i>Molecular Cell</i></b>, 55(5), 708-722.
        [<a href="https://doi.org/10.1016/j.molcel.2014.07.005">Full Text</a>],
        [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/NFY.pdf">PDF</a>]
        <br><br>
        
        Kumar, D., Cinghu, S., Oldfield, A., <b><u>Yang, P.</b></u> & Jothi, R. (2021)
        Decoding the function of bivalent chromatin in development and cancer.
        <b><i>Genome Research</i></b>, 31(12), 2170-2184.
        [<a href="https://doi.org/10.1101/gr.275736.121">Full Text</a>]
        <br><br>

        Cinghu, S.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Kosak, J., Conway, A., Kumar, D., Oldfield, A., Adelman, K. & Jothi, R. (2017)
        Intragenic enhancers attenuate host gene expression.
        <b><i>Molecular Cell</i></b>, 68(1), 104–117.
        [<a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30658-5">Full Text</a>],
        [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Attenuator.pdf">PDF</a>]
        <br><br>

    </div>
    
</div>
<br>


</div>
















