---
layout: default
---


<div class="container">
    <h3>Tools and resource related to single-cell omics data</h3>
    

    <!-- Cepo -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/Cepo_logo.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h4><font color="#8B0000">Cepo</font></h4>
            A new method to detect cell identity genes from single-cell RNA-sequencing data using differential stability, a biologically motivated metric. Cepo computes cell-type specific gene statistics pertaining to differential stable gene expression.
            <br><br>
            [<a href="https://pyanglab.github.io/Cepo/">Github Repo</a>], 
            [<a href="https://bioconductor.org/packages/release/bioc/html/Cepo.html">BioC R package</a>], 
            [<a href="https://pyanglab.github.io/Cepo/articles/cepo.html">User tutorial</a>]
            <br><br>
            
            <b>Kim, H.</b>, Wang, K., <b>Chen, C.</b>, Lin, Y., Tam, PPL., Lin, D., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Uncovering cell identity through differential stability with Cepo.
            <b><i>Nature Computational Science</i></b>, 1, 784-790.
            [<a href="https://doi.org/10.1038/s43588-021-00172-2">Full Text</a>]
            [<a href="https://rdcu.be/cDyXi">Nature Content Sharing link</a>]
            <br><br>
            
            <b>Kim, H.</b>, Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Defining cell identity beyond the premise of differential gene expression.
            <b><i>Cell Regeneration</i></b>, 10, 20.
            [<a href="https://doi.org/10.1186/s13619-021-00083-7">Full Text</a>]
			<br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- CiteFuse -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/CiteFuse.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h4><font color="#8B0000">CiteFuse</font></h4>
            A suite of methods and tools for CITE-seq data analysis from pre-processing to integrative analytics, including doublet detection, network-based modality integration, cell type clustering, differential RNA and protein expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualisation of the analyses.
            <br><br>
            [<a href="https://sydneybiox.github.io/CiteFuse/">Github Repo</a>], 
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>], 
            [<a href="https://sydneybiox.github.io/CiteFuse/articles/CiteFuse.html">User tutorial</a>]
            <br><br>
            
            <b>Kim, H.</b><sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            CiteFuse enables multi-modal analysis of CITE-seq data.
            <b><i>Bioinformatics</i></b>, 36(14), 4137-4143.
            [<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- scReClassify -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/scReClassify.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h4><font color="#8B0000">scReClassify</font></h4>
            scReClassify is a method for correcting potentially mislabelled cells in single-cell RNA-sequencing data. It uses a semi-supervised algorithm, adaSampling, to learn and predict mis-annotation of cell types in the dataset.
            <br><br>
            [<a href="https://github.com/SydneyBioX/scReClassify">Github Repo & tutorial</a>], 
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/scReClassify.html">BioC R package</a>]
            <br><br>
            
			<b>Kim, T.</b>, Lo, K., <b>Geddes, T.</b>, <b>Kim, H.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			scReClassify: post hoc cell type classification of single-cell RNA-seq data.
			<b><i>BMC Genomics</i></b>, 20, 913.
			[<a href="https://doi.org/10.1186/s12864-019-6305-x">Full Text</a>]
			<br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- SEGs -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h4><font color="#8B0000">Stably expressed genes</font></h4>
            Stably expressed genes (SEGs) are genes that express at similar levels across individual cells in a large number of cell types. They are identified from large-scale mouse and human sinlge-cell RNA-seq datasets covering a large number of cell types. SEGs can be used for batch correction and integration of multiple datasets. Biologically, SEGs share similar characteristics of housekeeping genes.
            <br><br>
            [<a href="/resources/scHK_mouse.xlsx">List of mouse SEGs</a>], 
            [<a href="/resources/scHK_human.xlsx">List of human SEGs</a>], 
            [<a href="http://shiny.maths.usyd.edu.au/SEGs">Interactive tool for SEG download</a>]
            <br><br>
            
    		Lin, Y., Ghazanfar, S., Strbenac, D., Wang, A., Patrick, E., Lin, D., Speed, T., Yang, J.<sup>&dagger;</sup> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
    		Evaluating stably expressed genes in single cells.
    		<b><i>GigaScience</i></b>, 8(9), giz106.
            [<a href="https://doi.org/10.1093/gigascience/giz106">Full Text</a>]
    		[<a href="http://gigadb.org/dataset/100637">Giga DB</a>]
    		<br><br>
        </div>
        <br>
    </div>
    <br>
    
    
    <h3>Tools and resource related to proteomics and phosphoproteomics data</h3>
    
    <!-- PhosR -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/PhosR.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h4><font color="#8B0000">PhosR</font></h4>
            An R package for comprehensive analysis of phosphoproteomic data. PhosR consists of various processing tools for phosphoproteomic data including filtering, imputation, normalisaton and batch correction using stably phosphorylated sites (SPS), which enables integration of multiple phosphoproteomic datasets. Downstream analytical tools consists of site- and protein-centric pathway analysis to evaluate activities of kinases and signalling pathways, large-scale kinase-substrate annotation from dynamic phosphoproteomic profiling, and visualisation and construction of signalomes present in the phosphoproteomic data of interest.
            <br><br>
            [<a href="https://github.com/PYangLab/PhosR">Github Repo</a>], 
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/PhosR.html">BioC R package</a>], 
            [<a href="https://pyanglab.github.io/PhosR/">User tutorial</a>], 
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
            
            <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            PhosR enables processing and functional analysis of phosphoproteomic data.
            <b><i>Cell Reports</i></b>, 34(8), 108771.
            [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
            <br><br>
            
            <b>Kim, H.</b>, <b>Kim, T.</b>, <b>Xiao, D.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR.
            <b><i>STAR Protocols</i></b>, 2(2), 100585.
            [<a href="https://doi.org/10.1016/j.xpro.2021.100585">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- Stable phosphoproteome -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h4><font color="#8B0000">Stably phosphorylated sites</font></h4>
            This list of stably phosphorylated sites (SPS) is identified from across 53 human phosphoproteomics datasets, covering 40 cell/tissue types and 194 conditions/treatments. SPS can be used for batch correction, normalisation, and integration of multiple phosphoproteomics datasets as in PhosR. The SPS themselves were evolutionarily conserved, functionally important, and enriched in a range of core signalling and gene pathways, including RNA splicing pathway, an essential cellular process in mammalian cells, and frequently disrupted by cancer mutations.
            <br><br>
            [<a href="/resources/human SPSs.xlsx">List of human SPS</a>], 
            [<a href="https://github.com/PYangLab/SPSs">Github Repo</a>]
            <br><br>
            
            <b>Xiao, D.</b>, <b>Kim, H.</b>, Pang, I. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2022)
            Functional analysis of the stable phosphoproteome reveals cancer vulnerabilities.
            <b><i>Bioinformatics</i></b>, 38(7), 1956-1963.
            [<a href="https://doi.org/10.1093/bioinformatics/btac015">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- ClueR -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/ClueR.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h4><font color="#8B0000">ClueR</font></h4>
            Cluster evaluation R (ClueR) package is designed for identifying optimal fuzzy c-means clustering of a given mass spectrometry (MS)-based phosphoproteomics data based on prior knowledge of kinase substrate relationships such as those from PhosphoSitePlus database.
            <br><br>
            [<a href="https://github.com/PYangLab/ClueR">Github Repo & tutorial</a>], 
            [<a href="https://CRAN.R-project.org/package=ClueR">CRAN R package</a>]
            <br><br>
            
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zheng, X., Jayaswal, V., Hu, G., Yang, J. & Jothi, R. (2015). 
			Knowledge-based analysis for detecting key signaling events from time-series phosphoproteomics data. 
			<b><i>PLoS Computational Biology</i></b>, 11(8), e1004403.
			[<a href="https://doi.org/10.1371/journal.pcbi.1004403">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/knowledge-based kinase detection.pdf">PDF</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- kinase substrate prediction -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/PUEL.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h4><font color="#8B0000">KSP-PUEL</font></h4>
            Positive-unlabeled ensemble learning (PUEL) for kinase substrate prediction (KSP-PUEL) is an application developed for predicting novel substrates of kinases of interest using MS-based phosphoproteomics data by learning from kinase recognition motifs and substrate phosphorylation profiles.
            <br><br>
            [<a href="https://github.com/PYangLab/KSP-PUEL">Github Repo</a>]
            <br><br>
            
            <b><u>Yang, P.</b></u><sup>&dagger;</sup>, Humphrey, S., James, D., Yang, J. & Jothi, R.<sup>&dagger;</sup> (2016). 
			Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data.
			<b><i>Bioinformatics</i></b>, 32(2), 252-259.
       		[<a href="https://doi.org/10.1093/bioinformatics/btv550">Full Text</a>], 
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Bioinformatics2016positiveUnlabelLearning.pdf">PDF</a>]
       		<br><br>
        </div>
        <br>
    </div>
    <br>

    <!-- directPA -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h4><font color="#8B0000">directPA & KinasePA</font></h4>
            An R package designed to identify combinatorial effects of multiple treatments and/or perturbations on pathways and kinases profiled by microarray, RNA-seq, proteomics, or phosphoproteomics data.
            <br><br>
            [<a href="https://github.com/PYangLab/directPA">Github Repo & tutorial</a>], 
            [<a href="https://cran.r-project.org/package=directPA">CRAN R package</a>]
            <br><br>
            
			<b><u>Yang, P.</b></u><sup>&#10018;</sup>, Patrick, E.<sup>&#10018;</sup>, Tan, S., Fazakerley, D., Burchfield, J., Gribben, C., Prior, M., James, D. & Yang, J. (2014). 
			Direction pathway analysis of large-scale proteomics data reveals novel features of the insulin action pathway. 
			<b><i>Bioinformatics</i></b>, 30(6), 808-814.
			[<a href="https://doi.org/10.1093/bioinformatics/btt616">Full Text</a>], 
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Direction.pdf">PDF</a>]
			<br><br>
            
            <b><u>Yang, P.</b></u>, Patrick, E., Humphrey, S., Ghazanfar, S., James, D., Jothi, R. & Yang, J. (2016). 
			KinasePA: Phosphoproteomics data annotation using hypothesis driven kinase perturbation analysis.
			<b><i>Proteomics</i></b>, 16(13), 1868-1871.
			[<a href="https://doi.org/10.1002/pmic.201600068">Full Text</a>]
			<br><br>
        </div>
        <br>
    </div>
    <br>
    
    
    <h3>General purpose tools</h3>
    
    <!-- AdaSampling -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h4><font color="#8B0000">AdaSampling</font></h4>
            AdaSampling is a semi-supervised model for learning from data with noisy class labels or only positive class. It can be applied to scenarios where there are mislabelled training examples in the data or only positive training examples are known whereas negative examples are unknown.
            <br><br>
            [<a href="https://CRAN.R-project.org/package=AdaSampling">CRAN package</a>], 
            [<a href="https://github.com/PengyiYang/AdaSampling">Github Repo</a>]
            <br><br>
            
            <b><u>Yang, P.</u></b><sup>&dagger;</sup>, Ormerod, J., Liu, W., <b>Ma, C.</b>, Zomaya, A. & Yang, J. (2019) 
			AdaSampling for positive-unlabeled and label noise learning with bioinformatics applications.
			<b><i>IEEE Transactions on Cybernetics</b></i>, 49(5), 1932-1943.
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/AdaSampling_v11.pdf">PDF</a>]
			<br><br>
            
            <b><u>Yang, P.</u></b>, Liu, W. & Yang, J. (2017) 
			Positive unlabeled learning via wrapper-based adaptive sampling.
			In <i>Proceedings of the 26th International Joint Conference on Artificial Intelligence <b>(IJCAI)</b></i>, 
			Pre-print [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/IJCAI17_Yang.pdf">PDF</a>]
			<br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- SSO -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h4><font color="#8B0000">Sample Subset Optimization (SSO)</font></h4>
            Sample subset optimization (SSO) is a sampling technique that utilize an evolutionary algorithm to optimize sample subsets for learning from imbalanced dataset. The key idea is to select a subset of most representative training examples from the major class and combine them with the minor class to generate a balance data for training classification models. 
            <br><br>
            [<a href="http://www.maths.usyd.edu.au/u/pengyi/software/Sampling.html">Software</a>] 
            <br><br>
            
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Yoo, P., Fernando, J., Zhou, B., Zhang, Z. & Zomaya, A. (2014). 
			Sample subset optimization techniques for imbalanced and ensemble learning problems in bioinformatics applications. 
			<b><i>IEEE Transactions on Cybernetics</i></b>, 44(3), 445-455.
			[<a href="https://doi.org/10.1109/TCYB.2013.2257480">Full Text</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/SSO for imbalanced and ensemble learning.pdf">PDF</a>]
            <br><br>
            
			<b><u>Yang, P.</b></u>, Zhang, Z., Zhou, B. & Zomaya, A. (2011). 
			Sample subset optimization for classifying imbalanced biological data. 
			In <i>Proceedings of the 15th Pacific-Asia Conference on Knowledge Discovery and Data Mining <b>(PAKDD)</b></i>,Â 
			Lecture Notes in Artificial Intelligence 6635, Springer Berlin Heidelberg, 333-344.
			[<a href="https://doi.org/10.1007/978-3-642-20847-8_28">Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
	
	<h3>Legacy projects hosted on Google Code</h3>
    
    <div class="section row" style="background-color: #F9F9F9;">
        <br>
        <ul>
            <li>Legacy version of sample subset optimization package<br> 
            <a href="https://code.google.com/p/sample-subset-optimization/">https://code.google.com/p/sample-subset-optimization</a><br><br>
            </li>

            <li>A machine learning algorithm in Java for protein inference<br>
            <a href="http://code.google.com/p/re-fraction/">http://code.google.com/p/re-fraction</a><br><br>
            </li>

            <li>A boosted learning algorithm in Java for peptide filtering<br>
            <a href="http://code.google.com/p/self-boosted-percolator/">http://code.google.com/p/self-boosted-percolator</a><br><br>
            </li>

            <li>A parallel genetic algorithm in Java for SNP interaction detection<br>
            <a href="http://code.google.com/p/genetic-ensemble-snpx/">http://code.google.com/p/genetic-ensemble-snpx</a><br><br>
            </li>

            <li>An ensemble algorithm in Perl for SNP interaction filtering<br>
            <a href="http://code.google.com/p/ensemble-of-filters/">http://code.google.com/p/ensemble-of-filters</a><br><br>
            </li>

            <li>An open source mass spectrometry analysis pipeline in R<br>
            <a href="http://code.google.com/p/ocap/">http://code.google.com/p/ocap</a><br><br>
            </li>

            <li>A dynamic wavelet package in C/C++ for mass spectrum modeling<br>
            <a href="http://code.google.com/p/dywave/DyWave/">http://code.google.com/p/dywave/DyWave</a><br><br>
            </li>

            <li>A particle swarm optimisation algorithm in Java for imbalanced data sampling<br>
            <a href="http://code.google.com/p/imbalanced-data-sampling/">http://code.google.com/p/imbalanced-data-sampling</a><br><br>
            </li>	
        </ul>
	</div>
</div>

