---
layout: default
---

<div class="container">
     <h2>Software</h2>
     <br>
	 
	 <ul>
	 	<li><b>KinasePA</b> (<a href="http://shiny.maths.usyd.edu.au/KinasePA">http://shiny.maths.usyd.edu.au/KinasePA</a>)<br>
     	Kinase perturbation analysis (KinasePA) is a web tool that allows you to identify key kinases that are 
     	perturbed in two treatments compared to control conditions (such as basal or unstimulated conditions).
        <br><br>
        Description: <br>
     	The input data should be a csv file separated by comma. The rows of the data file are phosphorylation 
     	sites and the columns are treatment1 vs control and treatment2 vs control. The values of the data file 
     	should be log2 fold changes. Here is an example <a href="http://www.maths.usyd.edu.au/u/pengyi/software/KinasePA/HEK.csv">File</a>
     	<br><br>
     	KinasePA has also been incorporated into "directPA" R package. Install the package and find out more:<br>
     	install.packages("directPA")
     	<br><br>
     	Reference:<br>
     	Yang, P., Patrick, E., Humphrey, S., Ghazanfar, S., James, D., Jothi, R. & Yang, J. (2016). 
     	KinasePA: Phosphoproteomics data annotation using hypothesis driven kinase perturbation analysis. 
     	Proteomics, 16(13), 1868-1871
     	</li>
     	
     	<br>
     	<li><b>PUEL</b> (<a href="https://github.com/PengyiYang/KSP-PUEL">https://github.com/PengyiYang/KSP-PUEL</a>)<br>
     	PUEL is an implementation of positive-unlabeled ensemble learning model for kinase-substrate prediction using
     	kinase recognition motifs and dynamic phosphoproteomics data.
     	<br><br>
     	Prediction results for Akt, mTOR, AMPK, and ERK from different organisms using large-scale phosphoproteomics data
     	are available from <a href="http://www.maths.usyd.edu.au/u/pengyi/software/PUEL.html">here</a>.
     	<br><br>
     	Reference:<br>
     	Yang, P., Humphrey, S., James, D., Yang, J. & Jothi, R. (2016). 
     	Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data. 
     	Bioinformatics, 32(2), 252-259.
     	</li>
     	
     	<br>
     	<li><b>ClueR</b> (<a href="https://CRAN.R-project.org/package=ClueR">https://CRAN.R-project.org/package=ClueR</a>)<br>
     	CLUster Evaluation (CLUE) is a computational method for identifying optimal number of clusters in a given 
     	time-course dataset clustered by cmeans or kmeans algorithms and subsequently identify key kinases or 
     	pathways from each cluster. Its implementation in R is called ClueR. 
     	<br><br>
     	Reference:<br>
     	Yang, P., Zheng, X., Jayaswal, V., Hu, G., Yang, J. & Jothi, R. (2015). Knowledge-based analysis for detecting 
     	key signaling events from time-series phosphoproteomics data. PLoS Computational Biology, 11(8), e1004403.
     	</li>
     	
     	<br>
     	<li><b>directPA</b> (<a href="https://cran.r-project.org/package=directPA">https://cran.r-project.org/package=directPA</a>)<br>
     	Direction analysis is a set of tools designed to identify combinatorial effects of multiple treatments 
     	and/or perturbations on pathways and kinases profiled by microarray, RNA-seq, proteomics, or phosphoproteomics.
     	<br><br>
     	Reference:<br>
     	Yang, P., Patrick, E., Tan, S., Fazakerley, D., Burchfield, J., Gribben, C., Prior, M., James, D. & Yang, J. (2014). 
     	Direction pathway analysis of large-scale proteomics data reveals novel features of the insulin action pathway. 
     	Bioinformatics, 30(6), 808-814.
     	</li>
     	
 	</ul>
	
	
	<h2>Projects hosted on Google Code</h2>
	<ul>
		<li>Sample subset optimization package in Java for building imbalanced ensemble<br> 
		<a href="https://code.google.com/p/sample-subset-optimization/">https://code.google.com/p/sample-subset-optimization</a><br><br>
		</li>
		
		<li>A machine learning algorithm in Java for protein inference<br>
		<a href="http://code.google.com/p/re-fraction/">http://code.google.com/p/re-fraction</a><br><br>
		</li>
		
		<li>A boosted learning algorithm in Java for peptide filtering<br>
		<a href="http://code.google.com/p/self-boosted-percolator/">http://code.google.com/p/self-boosted-percolator</a><br><br>
		</li>
		
		<li>A parallel genetic algorithm in Java for SNP interaction detection<br>
		<a href="http://code.google.com/p/genetic-ensemble-snpx/">http://code.google.com/p/genetic-ensemble-snpx</a><br><br>
		</li>
		
		<li>An ensemble algorithm in Perl for SNP interaction filtering<br>
		<a href="http://code.google.com/p/ensemble-of-filters/">http://code.google.com/p/ensemble-of-filters</a><br><br>
		</li>
		
		
	</ul>
	
	<br>
	
	<h2>Other resources</h2>
	<ul>
		<li> Prediction results for Akt, mTOR, AMPK, and ERK from different organisms using large-scale phosphoproteomics data
     	are available from <a href="http://www.maths.usyd.edu.au/u/pengyi/software/PUEL.html">here</a>.</li>
     	
	</ul>
	
</div>

