---
layout: default
---


<div class="container">
    <h3>Tools and resource related to single-cell omics data</h3>
    <br>
    <!--
    <h4>Most of the software and packages in developing phase are hosted on 
        <a href="https://github.com/PYangLab/">https://github.com/PYangLab/</a></h4>
    <br>
    -->
	 
    <!--<h3>R packages</h3>-->
    
    <!-- Cepo -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/Cepo_logo.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3 class="projects">Cepo</h3>
            A new method to detect cell identity genes from single-cell RNA-sequencing data using differential stability, a biologically motivated metric. Cepo computes cell-type specific gene statistics pertaining to differential stable gene expression.
            <br><br>
            [<a href="https://pyanglab.github.io/Cepo/">Github Repo</a>], 
            [<a href="https://bioconductor.org/packages/release/bioc/html/Cepo.html">BioC R package</a>], 
            [<a href="https://pyanglab.github.io/Cepo/articles/cepo.html">User tutorial</a>]
            <br><br>
            
            <b>Kim, H.</b>, Wang, K., <b>Chen, C.</b>, Lin, Y., Tam, PPL., Lin, D., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Uncovering cell identity through differential stability with Cepo.
            <b><i>Nature Computational Science</i></b>, 1, 784-790.
            [<a href="https://doi.org/10.1038/s43588-021-00172-2">Full Text</a>]
            [<a href="https://rdcu.be/cDyXi">Nature Content Sharing link</a>]
            <br><br>
            
            <b>Kim, H.</b>, Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Defining cell identity beyond the premise of differential gene expression.
            <b><i>Cell Regeneration</i></b>, 10, 20.
            [<a href="https://doi.org/10.1186/s13619-021-00083-7">Full Text</a>]
			<br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- CiteFuse -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/CiteFuse.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3 class="projects">CiteFuse</h3>
            A suite of methods and tools for CITE-seq data analysis from pre-processing to integrative analytics, including doublet detection, network-based modality integration, cell type clustering, differential RNA and protein expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualisation of the analyses.
            <br><br>
            [<a href="https://sydneybiox.github.io/CiteFuse/">Github Repo</a>], 
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>], 
            [<a href="https://sydneybiox.github.io/CiteFuse/articles/CiteFuse.html">User tutorial</a>]
            <br><br>
            
            <b>Kim, H.</b><sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            CiteFuse enables multi-modal analysis of CITE-seq data.
            <b><i>Bioinformatics</i></b>, 36(14), 4137-4143.
            [<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- scReClassify -->
    
    
    <!-- SEGs -->
    
    
    
    <h3>Tools and resource related to proteomics and phosphoproteomics data</h3>
    <br>
    
    <!-- PhosR -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/PhosR.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3 class="projects">PhosR</h3>
            An R package for comprehensive analysis of phosphoproteomic data. PhosR consists of various processing tools for phosphoproteomic data including filtering, imputation, normalisaton and batch correction using stably phosphorylated sites (SPS), which enables integration of multiple phosphoproteomic datasets. Downstream analytical tools consists of site- and protein-centric pathway analysis to evaluate activities of kinases and signalling pathways, large-scale kinase-substrate annotation from dynamic phosphoproteomic profiling, and visualisation and construction of signalomes present in the phosphoproteomic data of interest.
            <br><br>
            [<a href="https://github.com/PYangLab/PhosR">Github Repo</a>], 
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/PhosR.html">BioC R package</a>], 
            [<a href="https://pyanglab.github.io/PhosR/">User tutorial</a>], 
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
            
            <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            PhosR enables processing and functional analysis of phosphoproteomic data.
            <b><i>Cell Reports</i></b>, 34(8), 108771.
            [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
            <br><br>
            
            <b>Kim, H.</b>, <b>Kim, T.</b>, <b>Xiao, D.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR.
            <b><i>STAR Protocols</i></b>, 2(2), 100585.
            [<a href="https://doi.org/10.1016/j.xpro.2021.100585">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- Stable phosphoproteome -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h3 class="projects">Stably phosphorylated sites</h3>
            This list of stably phosphorylated sites (SPS) is identified from across 53 human phosphoproteomics datasets, covering 40 cell/tissue types and 194 conditions/treatments. SPS can be used for batch correction, normalisation, and integration of multiple phosphoproteomics datasets as in PhosR. The SPS themselves were evolutionarily conserved, functionally important, and enriched in a range of core signalling and gene pathways, including RNA splicing pathway, an essential cellular process in mammalian cells, and frequently disrupted by cancer mutations.
            <br><br>
            [<a href="/resources/Supplementary Table 2.xlsx">List of SPS</a>], 
            [<a href="https://github.com/PYangLab/SPSs">Github Repo</a>]
            <br><br>
            
            <b>Xiao, D.</b>, <b>Kim, H.</b>, Pang, I. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2022)
            Functional analysis of the stable phosphoproteome reveals cancer vulnerabilities.
            <b><i>Bioinformatics</i></b>, 38(7), 1956-1963.
            [<a href="https://doi.org/10.1093/bioinformatics/btac015">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    <!-- ClueR -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/ClueR.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3 class="projects">ClueR</h3>
            Cluster evaluation R (ClueR) package is designed for identifying optimal fuzzy c-means clustering of a given mass spectrometry (MS)-based phosphoproteomics data based on prior knowledge of kinase substrate relationships such as those from PhosphoSitePlus database.
            <br><br>
            [<a href="https://github.com/PYangLab/ClueR">Github Repo & tutorial</a>], 
            [<a href="https://CRAN.R-project.org/package=ClueR">CRAN R package</a>]
            <br><br>
            
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zheng, X., Jayaswal, V., Hu, G., Yang, J. & Jothi, R. (2015). 
			Knowledge-based analysis for detecting key signaling events from time-series phosphoproteomics data. 
			<b><i>PLoS Computational Biology</i></b>, 11(8), e1004403.
			[<a href="https://doi.org/10.1371/journal.pcbi.1004403">Full Text</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/knowledge-based kinase detection.pdf">PDF</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
    
    
    <!-- KinasePA -->
    
    <!-- directPA -->
    
    <!-- kinase substrate prediction -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/PUEL.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3 class="projects">KSP-PUEL</h3>
            Positive-unlabeled ensemble learning (PUEL) for kinase substrate prediction (KSP-PUEL) is an application developed for predicting novel substrates of kinases of interest using MS-based phosphoproteomics data by learning from kinase recognition motifs and substrate phosphorylation profiles.
            <br><br>
            [<a href="https://github.com/PYangLab/KSP-PUEL">Github Repo & tutorial</a>], 
            <br><br>
            
            <b><u>Yang, P.</b></u><sup>&dagger;</sup>, Humphrey, S., James, D., Yang, J. & Jothi, R.<sup>&dagger;</sup> (2016). 
			Positive-unlabeled ensemble learning for kinase substrate prediction from dynamic phosphoproteomics data.
			<b><i>Bioinformatics</i></b>, 32(2), 252-259.
       		[<a href="https://doi.org/10.1093/bioinformatics/btv550">Full Text</a>], 
       		[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Bioinformatics2016positiveUnlabelLearning.pdf">PDF</a>]
       		<br><br>
        </div>
        <br>
    </div>
    <br>

    
	 <ul>
	 	
         
         <!--
	 	<li><font color="#8B0000"><b>CiteFuse</b></font> (<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>)<br>
     	A set of methods and tools for multi-model analysis of single-cell multi-omics data, CITE-seq.
     	<br><br>
     	Reference:<br>
			Kim, H.<sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J., & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
			CiteFuse enables multi-modal analysis of CITE-seq data.
			<b><i>Bioinformatics</i></b>, 10.1093/bioinformatics/btaa282.
			[<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
			[<a href="https://www.biorxiv.org/content/10.1101/854299v1">Biorxiv version</a>]
     	</li><br>-->

	 	<li><font color="#8B0000"><b>AdaSampling</b></font> (<a href="https://CRAN.R-project.org/package=AdaSampling">https://CRAN.R-project.org/package=AdaSampling</a>)<br>
     	Implements the adaptive sampling procedure, a framework for both positive unlabeled learning and learning with class label noise. 
     	<br><br>
     	Reference:<br>
			<b><u>Yang, P.</u></b><sup>&dagger;</sup>, Ormerod, J., Liu, W., Ma, C., Zomaya, A., Yang, J. (2018) 
			AdaSampling for positive-unlabeled and label noise learning with bioinformatics applications.
			<b><i>IEEE Transactions on Cybernetics</b></i>, doi:10.1109/TCYB.2018.2816984
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/AdaSampling_v11.pdf">PDF</a>],
			[<a href="https://github.com/PYangLab/AdaSampling">Repo</a>]
     	</li><br>
     	
	 	
     	
     	<li><font color="#8B0000"><b>directPA</b></font> (<a href="https://cran.r-project.org/package=directPA">https://cran.r-project.org/package=directPA</a>)<br>
     	Direction analysis is a set of tools designed to identify combinatorial effects of multiple treatments 
     	and/or perturbations on pathways and kinases profiled by microarray, RNA-seq, proteomics, or phosphoproteomics.
     	<br><br>
     	Reference:<br>
			<b><u>Yang, P.</b></u><sup>&#10018;</sup>, Patrick, E.<sup>&#10018;</sup>, Tan, S., Fazakerley, D., Burchfield, J., Gribben, C., Prior, M., James, D. & Yang, J. (2014). 
			Direction pathway analysis of large-scale proteomics data reveals novel features of the insulin action pathway. 
			<b><i>Bioinformatics</i></b>, 30(6), 808-814.
			[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24167158">Pubmed</a>],
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/Direction.pdf">PDF</a>]
     	</li>
	 </ul>
	 
	 <h3>Shiny apps</h3>
	 <ul>
	 
         <!--
	 	<li><font color="#8B0000"><b>QCMAP</b></font> (<a href="http://shiny.maths.usyd.edu.au/QCMAP/">http://shiny.maths.usyd.edu.au/QCMAP/</a>)<br>
	 	An interactive web tool for mass spectrometry instrument diagnosis and data quality control.
	 	<br><br>
		Reference:<br>
		Kim, T., Chen, I., Parker, B., Humphrey, S., Crossett, B., Cordwell, S., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J<sup>&dagger;</sup>. (2019).
		QCMAP: An Interactive Web-Tool for Performance Diagnosis and Prediction of LC-MS Systems.
		<b><i>Proteomics</b></i>, <a href="https://doi.org/10.1002/pmic.201900068">https://doi.org/10.1002/pmic.201900068</a>
	 	</li><br>
            -->
	 	
	 	<li><font color="#8B0000"><b>KinasePA</b></font> (<a href="http://shiny.maths.usyd.edu.au/KinasePA">http://shiny.maths.usyd.edu.au/KinasePA</a>)<br>
     	Kinase perturbation analysis (KinasePA) is a web tool that allows you to identify key kinases that are 
     	perturbed in two treatments compared to control conditions (such as basal or unstimulated conditions).
        <br><br>
        Description: <br>
     	The input data should be a csv file separated by comma. The rows of the data file are phosphorylation 
     	sites and the columns are treatment1 vs control and treatment2 vs control. The values of the data file 
     	should be log2 fold changes. Here is an example <a href="http://www.maths.usyd.edu.au/u/pengyi/software/KinasePA/HEK.csv">File</a>
     	<br><br>
     	KinasePA has also been incorporated into "directPA" R package. Install the package and find out more:<br>
     	install.packages("directPA")
     	<br><br>
     	Reference:<br>
     	Yang, P., Patrick, E., Humphrey, S., Ghazanfar, S., James, D., Jothi, R. & Yang, J. (2016). 
     	KinasePA: Phosphoproteomics data annotation using hypothesis driven kinase perturbation analysis. 
     	<b><i>Proteomics</b></i>, 16(13), 1868-1871
     	[<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5027648/">full text</a>]
     	</li>
     </ul>
     	
     <h3>Standalone</h3>
     <ul>
     	<li><font color="#8B0000"><b>SSO</b></font> (<a href="http://www.maths.usyd.edu.au/u/pengyi/software/Sampling.html">http://www.maths.usyd.edu.au/u/pengyi/software/Sampling.html</a>)<br>
     	Sample subset optimization (SSO) is a sampling technique that utilize an evolutionary algorithm to optimize sample subsets for learning from 
     	imbalanced dataset. Please see more details in the reference below.
     	<br><br>
     	Reference:<br>
     	<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Yoo, P., Fernando, J., Zhou, B., Zhang, Z. & Zomaya, A. (2014). 
			Sample subset optimization techniques for imbalanced and ensemble learning problems in bioinformatics applications. 
			<b><i>IEEE Transactions on Cybernetics</i></b>, 44(3), 445-455.
			[<a href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6615954&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6615954">IEEE Xplore</a>]
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/SSO for imbalanced and ensemble learning.pdf">PDF</a>]
 	</ul>
	
	
	<h2>Projects hosted on Google Code</h2>
	<ul>
		<li>Legacy version of sample subset optimization package<br> 
		<a href="https://code.google.com/p/sample-subset-optimization/">https://code.google.com/p/sample-subset-optimization</a><br><br>
		</li>
		
		<li>A machine learning algorithm in Java for protein inference<br>
		<a href="http://code.google.com/p/re-fraction/">http://code.google.com/p/re-fraction</a><br><br>
		</li>
		
		<li>A boosted learning algorithm in Java for peptide filtering<br>
		<a href="http://code.google.com/p/self-boosted-percolator/">http://code.google.com/p/self-boosted-percolator</a><br><br>
		</li>
		
		<li>A parallel genetic algorithm in Java for SNP interaction detection<br>
		<a href="http://code.google.com/p/genetic-ensemble-snpx/">http://code.google.com/p/genetic-ensemble-snpx</a><br><br>
		</li>
		
		<li>An ensemble algorithm in Perl for SNP interaction filtering<br>
		<a href="http://code.google.com/p/ensemble-of-filters/">http://code.google.com/p/ensemble-of-filters</a><br><br>
		</li>
		
		<li>An open source mass spectrometry analysis pipeline in R<br>
		<a href="http://code.google.com/p/ocap/">http://code.google.com/p/ocap</a><br><br>
		</li>
		
		<li>A dynamic wavelet package in C/C++ for mass spectrum modeling<br>
		<a href="http://code.google.com/p/dywave/DyWave/">http://code.google.com/p/dywave/DyWave</a><br><br>
		</li>
			
		<li>A particle swarm optimisation algorithm in Java for imbalanced data sampling<br>
		<a href="http://code.google.com/p/imbalanced-data-sampling/">http://code.google.com/p/imbalanced-data-sampling</a><br><br>
		</li>	
	</ul>


	<h2>Other resources</h2>
	<ul>
     	<li> PhD thesis on ENSEMBLE METHODS AND HYBRID ALGORITHMS FOR COMPUTATIONAL AND SYSTEMS BIOLOGY (Pengyi Yang) [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/thesis/Pengyi-phdthesis-filed.pdf">PDF</a>]</li>
	</ul>
	
</div>

