---
layout: default
---

<div class="container">

    <h2>Teaching</h2>
     
    <!-- DATA3888 guest lecture -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h3><font color="#8B0000">Data Science Capstone (DATA3888), guest lecturer (2019 - present)</font></h3>
            I guest lecturing on ensemble, semi-supervised, and deep learning in unit DATA3888, Data Science Capstone.
            <br><br>
            
            <b>Lecture codes</b>
            <li> <a href=""></a>Ensemble learning</li>
            <li> <a href=""></a>Semi-supervised learning</li>
            <br><br>
            
            <b>References</b><br><br>
            <b><u>Yang, P.</u></b><sup>&dagger;</sup>, Ormerod, J., Liu, W., Ma, C., Zomaya, A., Yang, J. (2018) 
			AdaSampling for positive-unlabeled and label noise learning with bioinformatics applications.
			<b><i>IEEE Transactions on Cybernetics</b></i>, doi:10.1109/TCYB.2018.2816984
			[<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/AdaSampling_v11.pdf">PDF</a>],
			[<a href="https://github.com/PengyiYang/AdaSampling">Repo</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>        
    
    <!-- QBIO2001 -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h3><font color="#8B0000">Molecular Systems Biology (QBIO2001), lecturer (2018 - present)</font></h3>
            I'm a lecturer of Molecular Systems Biology (QBIO2001) jointly offered by the School of Mathematics and Statistics, and Life and Environmental Sciences.
            <br><br>
            
            <b>Lecture codes</b>
     			<li>Biological network reconstruction and visualisation (<a href="http://www.maths.usyd.edu.au/u/pengyi/teaching/Clustering.html">html</a>, <a href="http://www.maths.usyd.edu.au/u/pengyi/teaching/Clustering.Rmd">Rmd</a>)</li>
     			<li>Biological network analysis and visualisation (<a href="http://www.maths.usyd.edu.au/u/pengyi/teaching/Kmeans.html">html</a>, <a href="http://www.maths.usyd.edu.au/u/pengyi/teaching/Kmeans.Rmd">Rmd</a>)</li>
            <br><br>
            
            <b>References</b><br><br>
			<b><u>Yang, P.</b></u><sup>&dagger;</sup>, Zheng, X., Jayaswal, V., Hu, G., Yang, J. & Jothi, R. (2015). 
			Knowledge-based analysis for detecting key signaling events from time-series phosphoproteomics data. 
			<b><i>PLoS Computational Biology</i></b>, 11(8), e1004403.
			[<a href="https://doi.org/10.1371/journal.pcbi.1004403">Full Text</a>],
            [<a href="https://github.com/PYangLab/ClueR">Github Repo</a>],
            [<a href="https://CRAN.R-project.org/package=ClueR">CRAN R package</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>       
    
    <!-- STAT5003 -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="col-md-12">
            <h3><font color="#8B0000">Computational Statistical Methods (STAT5003), lecturer (2016 - 2018)</font></h3>
            I was the lecturer of Computational Statistical Methods (STAT5003), from 2016 - 2018, offered by the School of Mathematics and Statistics. Find more details regarding this unit <a href="https://www.sydney.edu.au/units/STAT5003">here</a>.
        </div>
        <br>
    </div>
    <br>
    
    
    <h2>Workshops & Tutorials</h2>
    
<!--    
    <h3>Single-cell data analysis</h3>
    <ul>
        <li><b>CITE-seq data analysis using CiteFuse</b> (<a href="https://sydneybiox.github.io/CiteFuse/index.html">https://sydneybiox.github.io/CiteFuse/index.html</a>)
        <br>
        CiteFuse package is designed for joint analysis of RNA and ADT data captured by CITE-seq technique in single cells.
        </li><br>
        
            <b>Cepo tutorials</b>
            Cepo is a method for detecting cell identity genes from single-cell RNA-seq data.
            <br>
            <li> <a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">PhosR STAR Protocol</a></li>
            <li> <a href="https://pyanglab.github.io/PhosR/">PhosR Vignette</a></li>
            <li> <a href="https://sydneybiox.github.io/BIS2019_PhosR/index.html">PhosR BioinfoSummer 2019 tutorial</a> </li>
            <br><br>
    </ul>
    <br>
-->    
    
    <!-- Single Cell Plus -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/Cepo.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3><font color="#8B0000">Single Cell Plus, a generic pipeline</font></h3>
            This tutorial introduces a collection of methods and tools our group has developed for analysing single-cell omics data.
            <br><br>
            
            <b>Single Cell Plus toturial</b>
            <li> <a href="https://sydneybiox.github.io/BIS2019_SC/index.html">Single Cell Plus, BioinfoSummer 2019</a></li>
            <br><br>
        </div>
        <br>
    </div>
    <br>    
    
    <!-- PhosR -->
    <div class="section row" style="background-color: #F9F9F9;">
        <div class="image col-md-1">
            <img class="img-responsive" src="/resources/image/PhosR.png"/ Align="Left">
        </div>
        <div class="col-md-11">
            <h3><font color="#8B0000">Phosphoproteomics data analysis using PhosR</font></h3>
            PhosR is an R package designed for phosphoproteomics data analysis. The PhosR repo provides the installation of the package and contains examples and tutorials for common analysis steps performed for phosphoproteomics datasets.
            <br><br>
        
            <b>PhosR tutorials</b>
            <li> <a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">PhosR STAR Protocol</a></li>
            <li> <a href="https://pyanglab.github.io/PhosR/">PhosR Vignette</a></li>
            <li> <a href="https://sydneybiox.github.io/BIS2019_PhosR/index.html">PhosR BioinfoSummer 2019 tutorial</a> </li>
            <br><br>
            
            <b>PhosR package repositories</b>
            <li> <a href="https://github.com/PYangLab/PhosR">PhosR Github Repo</a></li>
            <li> <a href="https://bioconductor.org/packages/PhosR/">PhosR Bioconductor</a></li>   
            <br><br>
            
            <b>PhosR references</b><br><br>
            <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            PhosR enables processing and functional analysis of phosphoproteomic data.
            <b><i>Cell Reports</i></b>, 34(8), 108771.
            [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
            <br><br>
            
            <b>Kim, H.</b>, <b>Kim, T.</b>, <b>Xiao, D.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR.
            <b><i>STAR Protocols</i></b>, 2(2), 100585.
            [<a href="https://doi.org/10.1016/j.xpro.2021.100585">Full Text</a>]
            <br><br>
        </div>
        <br>
    </div>
    <br>
     
</div>

