---
layout: default
---

<div class="container">
      <h2>Notations:</h2>
      <br>
            <b>Bold</b>: CSB group member<br>
    		&#10018;: Co-first author<br>
    		&dagger;: Corresponding/Co-corresponding author<br>
    
      <hr>

	<h2 class="publications">Thomas Geddes publications as part of computational systems biology group</h2>
    <br>
            6.
            Kearney, A., Norris, D., Ghomlaghi, M., Wong, M., Humphrey, S., Carroll, L., Yang, G., Cooke, K., <b><u>Yang, P.</b></u>, <b>Geddes, T.</b>, Shin, S., Fazakerley, D., Nguyen, L., James, D. & Burchfield, J. (2021)
            Akt phosphorylates insulin receptor substrate to limit PI3K-mediated PIP3 synthesis.
            <b><i>eLife</i></b>, 10, e66942.
            [<a href="https://doi.org/10.7554/eLife.66942">Full Text</a>]
			<br><br>

            5.
            Norris, D.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Shin, S.<sup>&#10018;</sup>, Kearney, A., <b>Kim, H.</b>, <b>Geddes, T.</b>, Senior A., Fazakerley, D., Nguyen, L., James, D. & Burchfield, J. (2021)
            Signalling heterogeneity is defined by pathway architecture and inter-cellular variability in protein expression.
            <b><i>iScience</i></b>, 24(2), 102118.
            [<a href="https://doi.org/10.1016/j.isci.2021.102118">Full Text</a>]
            <br><br>

            4.
            Cao, Y., <b>Geddes, T.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            Ensemble deep learning in bioinformatics.
            <b><i>Nature Machine Intelligence</i></b>, 2, 500-508.
            [<a href="https://doi.org/10.1038/s42256-020-0217-y">Full Text</a>]
            [<a href="https://rdcu.be/b6jIn">Nature Content Sharing link</a>]
            <br><br>
    
            3.
            <b>Kim, H.</b><sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
            CiteFuse enables multi-modal analysis of CITE-seq data.
            <b><i>Bioinformatics</i></b>, 36(14), 4137-4143.
            [<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
            [<a href="https://www.biorxiv.org/content/10.1101/854299v1">Biorxiv version</a>]
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>]
            <br><br>

			2. 
			<b>Geddes, T.</b>, <b>Kim, T.</b>, Nan, L., Burchfield, J., Yang, J., Tao, D. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis.
			<b><i>BMC Bioinformatics</i></b>, 20, 660.
			[<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5">Full Text</a>]
			[<a href="https://github.com/gedcom/scCCESS">Repo</a>]
			<br><br>

			1. 
			<b>Kim, T.</b>, Lo, K., <b>Geddes, T.</b>, <b>Kim, H.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			scReClassify: post hoc cell type classification of single-cell RNA-seq data.
			<b><i>BMC Genomics</i></b>, 20, 913.
			[<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6305-x">Full Text</a>]
			[<a href="https://github.com/SydneyBioX/scReClassify">Repo</a>]
			<br><br>
</div>
