---
layout: default
---

<div class="container">
      <h2>Notations:</h2>
      <br>
            <b>Bold</b>: CSB group member<br>
    		&#10018;: Co-first author<br>
    		&dagger;: Corresponding/Co-corresponding author<br>
    
      <hr>

	<h2 class="publications">Taiyun Kim's publications as part of the computational systems biology group</h2>
    <br>
            8.
            <b>Kim, H.</b>, <b>Kim, T.</b>, <b>Xiao, D.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR.
            <b><i>STAR Protocols</i></b>, 2(2), 100585.
            [<a href="https://doi.org/10.1016/j.xpro.2021.100585">Full Text</a>]
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
    
            7.
            <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            PhosR enables processing and functional analysis of phosphoproteomic data.
            <b><i>Cell Reports</i></b>, 34(8), 108771.
            [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
            [<a href="https://pyanglab.github.io/PhosR/">BioC R package</a>]
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
    
            6.
            <b>Kim, T.</b>, Tang, O., Vernon, S., Kott, K., Koay, Y., Park, J., James, D., Grieve, S., Speed, T., <b><u>Yang, P.</u></b><sup>&dagger;</sup>, Figtree, G.<sup>&dagger;</sup>, O'Sullivan, J.<sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2021)
            A hierarchical approach to removal of unwanted variation for large-scale metabolomics data.
            <b><i>Nature Communications</i></b>, 12, 4992.
            [<a href="https://doi.org/10.1038/s41467-021-25210-5">Full Text</a>]
            [<a href="https://github.com/SydneyBioX/hRUV">Repo</a>]
            <br><br>    

			5. 
			<b>Geddes, T.</b>, <b>Kim, T.</b>, Nan, L., Burchfield, J., Yang, J., Tao, D. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			Autoencoder-based cluster ensembles for single-cell RNA-seq data analysis.
			<b><i>BMC Bioinformatics</i></b>, 20, 660.
			[<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3179-5">Full Text</a>]
			[<a href="https://github.com/gedcom/scCCESS">Repo</a>]
			<br><br>

			4. 
			<b>Kim, T.</b>, Lo, K., <b>Geddes, T.</b>, <b>Kim, H.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			scReClassify: post hoc cell type classification of single-cell RNA-seq data.
			<b><i>BMC Genomics</i></b>, 20, 913.
			[<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6305-x">Full Text</a>]
			[<a href="https://github.com/SydneyBioX/scReClassify">Repo</a>]
			<br><br>

			3.
			<b>Kim, T.</b>, <b>Chen, I.</b>, Parker, B., Humphrey, S., Crossett, B., Cordwell, S., <b><u>Yang, P.</u></b><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2019)
			QCMAP: An interactive web-tool for performance diagnosis and prediction of LC-MS Systems.
			<b><i>Proteomics</i></b>, 19(3), 1900068.
			[<a href="https://doi.org/10.1002/pmic.201900068">Full Text</a>]
            [<a href="http://www.maths.usyd.edu.au/u/pengyi/publication/2019/QCMAP.pdf">PDF</a>]
			[<a href="http://shiny.maths.usyd.edu.au/QCMAP/">QCMAP tool</a>]
			<br><br>

    		2.
			<b>Kim, T.</b>, <b>Chen, I.</b>, Lin, Y., Wang, A., Yang, J. & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2019)
			Impact of similarity metrics on single-cell RNA-seq data clustering.
			<b><i>Briefings in Bioinformatics</i></b>, 20(6), 2316-2326.
			[<a href="https://doi.org/10.1093/bib/bby076">PDF</a>]
			[<a href="https://www.researchgate.net/publication/326769441_Impact_of_similarity_metrics_on_single-cell_RNA-seq_data_clustering">draft</a>]
			[<a href="https://github.com/SydneyBioX/scdney">Repo</a>]
    		<br><br>

    		1.
    		<b><u>Yang, P.</u></b><sup>&dagger;</sup>, Oldfield, A., <b><u>Kim, T.</u></b>, Yang, A., Yang, J. & Ho, J.<sup>&dagger;</sup> (2017)
    		Integrative analysis identifies co-dependent gene expression regulation of BRG1 and CHD7 at distal regulatory sites in embryonic stem cells.
			<b><i>Bioinformatics</i></b>, 33(13), 1916â€“1920.
			[<a href="https://academic.oup.com/bioinformatics/article/2995820/Integrative-analysis-identifies-co-dependent-gene">Full Text</a>], 
			[<a href="http://pad2.maths.usyd.edu.au/PADv1/">Interactive PAD</a>]
    		<br><br>
</div>
