---
layout: default
---

<div class="container">
      <h2>Notations:</h2>
      <br>
            <b>Bold</b>: CSB group member<br>
    		&#10018;: Co-first author<br>
    		&dagger;: Corresponding/Co-corresponding author<br>
    
      <hr>

	<h2 class="publications">Hani Kim's publications as part of computational systems biology group</h2>
	<br>

            12.
            <b>Xiao, D.</b>, Caldow, M., <b>Kim, H.</b>, Blazev, R., Koopman, R., Manandi, D., Parker, B. & <b><u>Yang, P.</u></b><sup>&dagger;</sup> (2022)
            Time-resolved phosphoproteome and proteome analysis reveals kinase signalling on master transcription factors during myogenesis.
            <b><i>iScience</i></b>, https://doi.org/10.1016/j.isci.2022.104489
            [<a href="https://doi.org/10.1016/j.isci.2022.104489">Full Text</a>]
            <br><br>
    
            11.
            Fernando, M., Lee, S., Wark, J., <b>Xiao, D.</b>, Lim, B., O'Hara-Wright, M., <b>Kim, H.</b>, Smith, G., Wong, T., Teber, E., Ali, R., <b><u>Yang, P.</u></b>, Graham, M. & Gonzalez-Cordero, A. (2022)
            Differentiation of brain and retinal organoids from confluent cultures of pluripotent stem cells connected by nerve-like axonal projections of optic origin.
            <b><i>Stem Cell Reports</i></b>, 
            [<a href="https://doi.org/doi:10.1016/j.stemcr.2022.04.003">Full Text</a>]
            <br><br>
            
            10.
            <b>Xiao, D.</b>, <b>Kim, H.</b>, Pang, I. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2022)
            Functional analysis of the stable phosphoproteome reveals cancer vulnerabilities.
            <b><i>Bioinformatics</i></b>, 38(7), 1956-1963.
            [<a href="https://doi.org/10.1093/bioinformatics/btac015">Full Text</a>]
            <br><br>
    
            9.
            <b>Kim, H.</b>, Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Defining cell identity beyond the premise of differential gene expression.
            <b><i>Cell Regeneration</i></b>, 10, 20.
            [<a href="https://doi.org/10.1186/s13619-021-00083-7">Full Text</a>]
			<br><br>
    
            8.
            Norris, D.<sup>&#10018;</sup>, <b><u>Yang, P.</b></u><sup>&#10018;</sup>, Shin, S.<sup>&#10018;</sup>, Kearney, A., <b>Kim, H.</b>, <b>Geddes, T.</b>, Senior A., Fazakerley, D., Nguyen, L., James, D. & Burchfield, J. (2021)
            Signalling heterogeneity is defined by pathway architecture and inter-cellular variability in protein expression.
            <b><i>iScience</i></b>, 24(2), 102118.
            [<a href="https://doi.org/10.1016/j.isci.2021.102118">Full Text</a>]
		    <br><br>

            7.
            <b>Kim, H.</b>, Wang, K., <b>Chen, C.</b>, Lin, Y., Tam, PPL., Lin, D., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Uncovering cell identity through differential stability with Cepo.
            <b><i>Nature Computational Science</i></b>, 1, 784-790.
            [<a href="https://doi.org/10.1038/s43588-021-00172-2">Full Text</a>]
            [<a href="https://rdcu.be/cDyXi">Nature Content Sharing link</a>]
            [<a href="https://bioconductor.org/packages/release/bioc/html/Cepo.html">BioC R package</a>]
            <br><br>

            6.
            <b>Kim, H.</b>, <b>Kim, T.</b>, <b>Xiao, D.</b> & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            Protocol for the processing and downstream analysis of phosphoproteomic data with PhosR.
            <b><i>STAR Protocols</i></b>, 2(2), 100585.
            [<a href="https://doi.org/10.1016/j.xpro.2021.100585">Full Text</a>]
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>
    
            5.
            <b>Kim, H.</b><sup>&#10018;</sup>, <b>Kim, T.</b><sup>&#10018;</sup>, Hoffman, N., <b>Xiao, D.</b>, James, D., Humphrey, S. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2021)
            PhosR enables processing and functional analysis of phosphoproteomic data.
            <b><i>Cell Reports</i></b>, 34(8), 108771.
            [<a href="https://doi.org/10.1016/j.celrep.2021.108771">Full Text</a>]
            [<a href="https://pyanglab.github.io/PhosR/">BioC R package</a>]
            [<a href="https://pyanglab.github.io/PhosR/articles/web/PhosR_STAR_protocols.html">STAR Protocol</a>]
            <br><br>

            4.
		<b>Kim, H.</b>, Osteil, P., Humphrey, S., Cinghu, S., Oldfield, A., Patrick, E., Wilkie, E., Peng, G., Suo, S., Jothi, R., Tam, P. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
		Transcriptional network dynamics during the progression of pluripotency revealed by integrative statistical learning.
		<b><i>Nucleic Acids Research</i></b>, 48(4), 1828-1842.
		[<a href="https://doi.org/10.1093/nar/gkz1179">Full Text</a>]
		<br><br>

            3.
            Lin, Y., Cao, Y., <b>Kim, H.</b>, Salim, A., Speed, T., Lin, D., <b><u>Yang, P.</b></u><sup>&dagger;</sup> & Yang, J.<sup>&dagger;</sup> (2020)
            scClassify: sample size estimation and multiscale classification of cells using single and multiple reference.
            <b><i>Molecular Systems Biology</i></b>, 16, e9389.
            [<a href="https://doi.org/10.15252/msb.20199389">Full Text</a>]
            [<a href="http://www.bioconductor.org/packages/release/bioc/html/scClassify.html">BioC R package</a>]
            <br><br>
    
		2.
		<b>Kim, H.</b><sup>&#10018;</sup>, Lin, Y.<sup>&#10018;</sup>, Geddes, T., Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2020)
		CiteFuse enables multi-modal analysis of CITE-seq data.
		<b><i>Bioinformatics</i></b>, 36(14), 4137-4143.
		[<a href="https://doi.org/10.1093/bioinformatics/btaa282">Full Text</a>]
		[<a href="https://www.biorxiv.org/content/10.1101/854299v1">Biorxiv version</a>]
		[<a href="http://www.bioconductor.org/packages/release/bioc/html/CiteFuse.html">BioC R package</a>]
		<br><br>

			1.
			<b>Kim, T.</b>, Lo, K., <b>Geddes, T.</b>, <b>Kim, H.</b>, Yang, J. & <b><u>Yang, P.</b></u><sup>&dagger;</sup> (2019)
			scReClassify: post hoc cell type classification of single-cell RNA-seq data.
			<b><i>BMC Genomics</i></b>, 20, 913.
			[<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-6305-x">Full Text</a>]
			[<a href="https://github.com/SydneyBioX/scReClassify">Repo</a>]
			<br><br>
</div>